

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Annotation data management &mdash; myProMS  documentation</title>
  

  
  
  
  

  
  <script type="text/javascript" src="../../_static/js/modernizr.min.js"></script>
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
        <script type="text/javascript" src="../../_static/jquery.js"></script>
        <script type="text/javascript" src="../../_static/underscore.js"></script>
        <script type="text/javascript" src="../../_static/doctools.js"></script>
        <script type="text/javascript" src="../../_static/language_data.js"></script>
    
    <script type="text/javascript" src="../../_static/js/theme.js"></script>

    

  
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/css/custom.css" type="text/css" />
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Server administration" href="server_administration.html" />
    <link rel="prev" title="Users management" href="users_management.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../../index.html" class="icon icon-home"> myProMS
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Getting Started</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/installation.html">Installation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../getting_started/installation.html#git-repository">Git Repository</a></li>
<li class="toctree-l2"><a class="reference internal" href="../getting_started/installation.html#docker-deployment">Docker deployment</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/login.html">Login</a></li>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/server_architecture.html">Server Architecture</a></li>
</ul>
<p class="caption"><span class="caption-text">Server Management</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="users_management.html">Users management</a><ul>
<li class="toctree-l2"><a class="reference internal" href="users_management.html#user-classes-and-access-privileges">User classes and access privileges</a></li>
<li class="toctree-l2"><a class="reference internal" href="users_management.html#account-management">Account management</a></li>
</ul>
</li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Annotation data management</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#sequence-databanks">Sequence databanks</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#databank-types">Databank types</a></li>
<li class="toctree-l3"><a class="reference internal" href="#listing-databanks">Listing databanks</a></li>
<li class="toctree-l3"><a class="reference internal" href="#adding-a-new-databank">Adding a new databank</a></li>
<li class="toctree-l3"><a class="reference internal" href="#editing-a-databank">Editing a databank</a></li>
<li class="toctree-l3"><a class="reference internal" href="#deleting-a-databank">Deleting a databank</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#spectral-swath-libraries-management">Spectral (SWATH) libraries management</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#listing-spectral-libraries">Listing spectral libraries</a></li>
<li class="toctree-l3"><a class="reference internal" href="#adding-a-new-library">Adding a new library</a></li>
<li class="toctree-l3"><a class="reference internal" href="#merging-two-library">Merging two library</a></li>
<li class="toctree-l3"><a class="reference internal" href="#editing-a-library">Editing a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="#updating-a-library">Updating a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="#restoring-the-previous-version-of-a-library">Restoring the previous version of a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="#searching-for-proteins-in-a-library">Searching for proteins in a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="#exporting-a-library">Exporting a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="#deleting-a-spectral-library">Deleting a spectral library</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#go-files-management">GO files management</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#ontology-files">Ontology files</a></li>
<li class="toctree-l3"><a class="reference internal" href="#annotation-files">Annotation files</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#species">Species</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#listing-species">Listing species</a></li>
<li class="toctree-l3"><a class="reference internal" href="#adding-or-editing-a-species">Adding or editing a species</a></li>
<li class="toctree-l3"><a class="reference internal" href="#deleting-a-species">Deleting a species</a></li>
<li class="toctree-l3"><a class="reference internal" href="#sequence-modifications">Sequence modifications</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#editing-or-merging-ptms">Editing or merging PTMs</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="server_administration.html">Server administration</a></li>
</ul>
<p class="caption"><span class="caption-text">Project management</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../project_management/project_settings.html">Projects settings</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#selection">Selection</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#creation">Creation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#accessibility">Accessibility</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#organization">Organization</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/project_settings.html#experiments"><strong>Experiments</strong></a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/project_settings.html#samples"><strong>Samples</strong></a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/project_settings.html#analyses"><strong>Analyses</strong></a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#navigation">Navigation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#life-span">Life span</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../project_management/analysis_import.html">Analysis import</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#supported-search-engines">Supported search engines</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#collecting-search-files">Collecting search files</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#import-parameters-mascot-proteome-discoverer-and-x-tandem">Import parameters (Mascot, Proteome Discoverer and X!Tandem)</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#importing-maxquant-data">Importing MaxQuant data</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#import-dia-data">Import DIA data</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#from-peakview">From PeakView</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#from-openswath">From OpenSWATH</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#running-openswath-quantification">Running OpenSWATH quantification</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#from-spectronaut">From Spectronaut</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#analysis-summary">Analysis summary</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../project_management/analysis_validation.html">Analysis validation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#automated-peptide-protein-validation">Automated peptide/protein validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#fdr-false-discovery-rate-based-validation">FDR (False discovery rate) - based validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#qualitative-validation">Qualitative validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#comparative-validation">Comparative validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#validation-templates">Validation templates</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#manual-peptide-protein-validation">Manual peptide/protein validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#peptides-selection-exclusion">Peptides selection/exclusion</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#protein-exclusion-and-filtering">Protein exclusion and filtering</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#lower-scoring-peptides-activation">Lower-scoring peptides activation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#clear-peptide-protein-selections">Clear peptide/protein selections</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#sequence-modification-validation">Sequence modification validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#phosphorylation-sites-validation-with-phosphors">Phosphorylation sites validation with PhosphoRS</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#manual-validation-of-modifications">Manual validation of modifications</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#validation-traceability">Validation traceability</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#reporting">Reporting</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../project_management/biological_samples.html">Biological Samples</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#properties">Properties</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#treatments">Treatments</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#recording-a-biological-sample">Recording a biological sample</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#linking-biological-samples-to-ms-analyses">Linking biological samples to MS Analyses</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Protein Identification</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/validated_proteins.html">Validated proteins</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/validated_proteins.html#match-groups-and-protein-visibility">Match groups and protein visibility</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#top-protein-selection-rules">Top protein selection rules</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#project-wide-protein-visibility-rules">Project-wide protein visibility rules</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#checking-for-conflicting-match-groups">Checking for conflicting match groups</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#displaying-match-group-composition">Displaying match group composition</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#manual-edition">Manual edition</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#peptide-distribution-in-match-group">Peptide distribution in match group</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/validated_proteins.html#identifier-mapping">Identifier mapping</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/validated_proteins.html#single-protein-view">Single protein view</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/protein_lists.html">Protein lists</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/protein_lists.html#project-based-protein-lists">Project-based protein lists</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/protein_lists.html#user-defined-protein-lists">User-defined protein lists</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/protein_lists.html#export-protein-lists">Export protein lists</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/search_proteins.html">Search for proteins</a></li>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/compare_proteins.html">Compare project proteins</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#full-protein-level-comparison">Full protein-level comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#pairwise-protein-level-comparison">Pairwise protein-level comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#pairwise-peptide-level-comparison">Pairwise peptide-level comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#saving-a-comparison">Saving a comparison</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Quantification</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../quantification/peptide_quantification.html">Peptide Quantification</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../quantification/peptide_quantification.html#data-import-from-search-results-file">Data import from search results file</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/peptide_quantification.html#data-extraction-from-lc-ms-file-with-masschroq">Data extraction from LC/MS file with MassChroQ</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../quantification/peptide_quantification.html#managing-mzxml-files">Managing mzXML files</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/peptide_quantification.html#running-xic-extraction">Running XIC extraction</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/peptide_quantification.html#virtual-ghost-peptides-and-proteins">Virtual/Ghost peptides and proteins</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/peptide_quantification.html#displaying-peptide-quantification-data">Displaying peptide quantification data</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../quantification/protein_quantification.html">Protein Quantification</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../quantification/protein_quantification.html#absolute-abundance-quantification">Absolute abundance quantification</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#empai-label-free">emPAI (label-free)</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#sin-label-free">SIn (label-free)</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#maxquant-intensity-lfq-ibaq">MaxQuant: Intensity, LFQ, iBAQ</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#displaying-single-abundance-quantification-data">Displaying single abundance quantification data</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/protein_quantification.html#relative-abundance-quantification">Relative abundance quantification</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#single-analysis-quantification-labeled">Single-Analysis quantification (labeled)</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#design-based-quantifications">Design-based quantifications</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#displaying-relative-abundance-quantification-data">Displaying relative abundance quantification data</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/protein_quantification.html#label-free-quantifications">Label-free quantifications</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#tnpq">TnPQ</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/protein_quantification.html#comparing-multiple-protein-quantifications">Comparing multiple protein quantifications</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/protein_quantification.html#exporting-multiple-quantifications">Exporting multiple quantifications</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../quantification/ptms_quantification.html">PTMs quantification</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../quantification/ptms_quantification.html#set-ptms-relevance-to-project">Set PTMs relevance to project</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/ptms_quantification.html#display-modification-sites-distribution">Display modification sites distribution</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/ptms_quantification.html#compare-ptms-between-projects">Compare PTMs between projects</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/ptms_quantification.html#quantify-modification-sites">Quantify modification sites</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Data Analysis</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../data_analysis/exploratory_analysis.html">Exploratory analysis</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#launching-exploratory-analyses-on-protein-quantification">Launching exploratory analyses on protein quantification</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#launching-exploratory-analyses-on-peptides-count">Launching exploratory analyses on peptides count</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#principal-component-analysis-pca">Principal Component Analysis (PCA)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#summary-edit-delete">Summary / edit / delete</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#displaying-a-pca">Displaying a PCA</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#d-clustering">2D-Clustering</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#id1">Summary / edit / delete</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#displaying-clustering">Displaying clustering</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../data_analysis/functional_analysis.html">Functional analysis</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/functional_analysis.html#gene-ontology">Gene Ontology</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#go-summary">GO summary</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#go-enrichment-analysis">GO enrichment analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#quantitative-gene-enrichment-analysis">Quantitative gene enrichment analysis</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/functional_analysis.html#pathway-enrichment">Pathway enrichment</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#launch-pathway-analysis">Launch pathway analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#summary-edit-delete">Summary / edit / delete</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#displaying-pathway-analysis">Displaying pathway analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#export">Export</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/functional_analysis.html#motif-enrichment-analysis">Motif enrichment analysis</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#launch-analysis">Launch analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#summary-editing-deleting">Summary/editing/deleting</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#displaying-motif-enrichment">Displaying motif enrichment</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#heatmap-motif-analysis">Heatmap motif analysis</a></li>
</ul>
</li>
</ul>
</li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">myProMS</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../index.html">Docs</a> &raquo;</li>
        
      <li>Annotation data management</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="../../_sources/docs/server_management/annotation_data.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="annotation-data-management">
<h1>Annotation data management<a class="headerlink" href="#annotation-data-management" title="Permalink to this headline">¶</a></h1>
<div class="section" id="sequence-databanks">
<h2>Sequence databanks<a class="headerlink" href="#sequence-databanks" title="Permalink to this headline">¶</a></h2>
<p>The sequence databanks used by the search engines must be referenced in <em>myProMS</em> so that protein annotations (identifier, description, species and sequence) sometimes not present in search result files (eg. <em>Mascot</em>) can be retrieved from the corresponding fasta file during analysis import.</p>
<p>A referenced sequence databank is also associated with a specific parse rule that allows <em>myProMS</em> to properly match and extract the annotation from the fasta file.</p>
<div class="section" id="databank-types">
<h3>Databank types<a class="headerlink" href="#databank-types" title="Permalink to this headline">¶</a></h3>
<p>Multiple databank types are available in <em>myProMS</em> depending on the proteomic resource used to download the fasta file, corresponding to <strong>different fasta file entries</strong>:</p>
<ul class="simple">
<li><dl class="first docutils">
<dt>UniProt:</dt>
<dd><span class="courier">&gt;sp|P15311|EZRI_HUMAN Ezrin OS=Homo sapiens (Human) GN=EZR Ezrin</span></dd>
</dl>
</li>
</ul>
<table border="1" class="align-center docutils">
<colgroup>
<col width="23%" />
<col width="25%" />
<col width="32%" />
<col width="20%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td>&#160;</td>
<td><strong>UniProt - ALL</strong></td>
<td><strong>UniProt - ACC / ACC#-n</strong></td>
<td><strong>UniProt - ID</strong></td>
</tr>
<tr class="row-even"><td>Protein identifier</td>
<td>sp|P15311|EZRI_HUMAN</td>
<td>P15311</td>
<td>EZRI_HUMAN</td>
</tr>
<tr class="row-odd"><td>Description</td>
<td colspan="3">Ezrin</td>
</tr>
<tr class="row-even"><td>Species</td>
<td colspan="3">Homo sapiens</td>
</tr>
</tbody>
</table>
<ul>
<li><dl class="first docutils">
<dt>SWISSPROT/trEMBL #1, #2 and #3</dt>
<dd><p class="first">These 3 types are equivalent of the 3 UniProt types described above except that they recognize the obsolete fasta entry format:</p>
<p class="last"><span class="courier">&gt;sp|P15311|EZRI_HUMAN Ezrin (p81) (Cytovillin) (Villin-2) - Homo sapiens (Human)</span></p>
</dd>
</dl>
</li>
<li><dl class="first docutils">
<dt>NCBI:</dt>
<dd><p class="first last"><span class="courier">&gt;gi|125987826|sp|P15311|EZRI_HUMAN Ezrin (p81) (Cytovillin) (Villin-2) [Homo sapiens (Human)]</span></p>
</dd>
</dl>
</li>
</ul>
<table border="1" class="docutils">
<colgroup>
<col width="29%" />
<col width="50%" />
<col width="21%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td>&#160;</td>
<td><strong>NCBInr - ALL</strong></td>
<td><strong>NCBI - GI</strong></td>
</tr>
<tr class="row-even"><td>Protein identifier</td>
<td>gi|125987826|sp|P15311|EZRI_HUMAN</td>
<td>gi|125987826</td>
</tr>
<tr class="row-odd"><td>Description</td>
<td colspan="2">Ezrin (p81) (Cytovillin) (Villin-2)</td>
</tr>
<tr class="row-even"><td>Species</td>
<td colspan="2">Homo sapiens</td>
</tr>
</tbody>
</table>
<ul class="simple">
<li><dl class="first docutils">
<dt>IPI:</dt>
<dd><span class="courier">&gt;IPI:IPI00843975.1|SWISS-PROT:P15311| Tax_Id=9606 Gene_Symbol=EZR Ezrin`</span></dd>
</dl>
</li>
</ul>
<table border="1" class="docutils">
<colgroup>
<col width="36%" />
<col width="64%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td>Protein identifier</td>
<td>IPI00843975</td>
</tr>
<tr class="row-even"><td>Description</td>
<td>Tax_Id=9606 Gene_Symbol=EZR Ezrin</td>
</tr>
<tr class="row-odd"><td>Species</td>
<td>9606</td>
</tr>
</tbody>
</table>
<div class="admonition warning">
<p class="first admonition-title">Warning</p>
<p class="last">The IPI resource is no longer maintained. We do not recommend using fasta files from this resource with your MS search engines.</p>
</div>
<ul class="simple">
<li><dl class="first docutils">
<dt>Undefined source: Protein (User-defined)</dt>
<dd>This type can be used as a temporary solution for any unknown or custom fasta-compatible entry: <span class="courier">&gt;pipe_separated_identifier_block any text</span></dd>
</dl>
</li>
</ul>
<table border="1" class="docutils">
<colgroup>
<col width="45%" />
<col width="55%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td>Protein identifier</td>
<td>Whole identifier block</td>
</tr>
<tr class="row-even"><td>Description</td>
<td>Everything else</td>
</tr>
<tr class="row-odd"><td>Species</td>
<td><strong>None recorded</strong></td>
</tr>
</tbody>
</table>
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">If you need you need to handle other entries type, please contact your local <em>myProMS</em> administrator or email to <code class="docutils literal notranslate"><span class="pre">myproms&#64;curie.fr</span></code>.</p>
</div>
</div>
<div class="section" id="listing-databanks">
<h3>Listing databanks<a class="headerlink" href="#listing-databanks" title="Permalink to this headline">¶</a></h3>
<p>Only bioinformatician and massists/managers with granted appropriate privileges can access and manage the protein databanks.
From <em>myProMS</em> main window, select <code class="docutils literal notranslate"><span class="pre">Annotation</span> <span class="pre">data</span></code> and follow the <code class="docutils literal notranslate"><span class="pre">Sequence</span> <span class="pre">databanks</span></code> link.
All active databanks are listed in alphabetic order with a short summary of information as shown in the screen capture below.
From this window, you can either <strong>add a new databank</strong>, <strong>edit</strong> or <strong>delete</strong> an existing one.</p>
<p>&lt;Figure of databanks list&gt;</p>
</div>
<div class="section" id="adding-a-new-databank">
<h3>Adding a new databank<a class="headerlink" href="#adding-a-new-databank" title="Permalink to this headline">¶</a></h3>
<p>From the databank list window, click on the <code class="docutils literal notranslate"><span class="pre">Add</span> <span class="pre">new</span> <span class="pre">databank</span></code> button at the top or bottom of the list.</p>
<p>The following form will be displayed:</p>
<p>&lt;Figure Add new Databank&gt;</p>
<p>Fill out the form to provide information on the databank you want to add. In particular, you must select the databank type so that the server will know how to extract the protein annotation from the file. Information on the corresponding parse rule is then displayed to help insure the right databank type was selected. You must also provide a <strong>fasta</strong> file containing the protein data.</p>
<p>There are multiple ways to do so:</p>
<ol class="arabic simple">
<li>Use a databank already referenced by Mascot: <em>myProMS</em> allows you to directly use fasta files stored on the Mascot server to avoid data duplication. In this case, the databank will be automatically synchronized when updated by Mascot.</li>
<li>Use a file from a dedicated directory on server (e.g. file was previously uploaded by FTP or the directory is shared between local computer and server).</li>
<li>Upload a fasta file from your computer.</li>
<li>Download the file from the internet: You must provide an HTTP or FTP link to the file.</li>
</ol>
<p>For the last 3 options, normal and gzip-/zip-compressed files are handled.
If the databank contains both target and decoy sequences, this must be specified as well as the decoy tag used (eg. REV_).</p>
<p>For the first 2 options (except if a compressed file is used in the 2nd option), it is possible to test the type of annotation rules selected before actually creating the new databank: Select a databank type, the file to be used and click on the <code class="docutils literal notranslate"><span class="pre">Test</span> <span class="pre">rules</span></code> button. Annotations from up to 10 entries from the file will be extracted using the selected rules and displayed. Select another set of rules and try again if the extraction did not match your expectations.</p>
<p>If the databank is species-specific, it is recommended to provide the species scientific name even if already specified in the protein entry lines of the fasta file.
Click on the <code class="docutils literal notranslate"><span class="pre">Save</span> <span class="pre">button</span></code> to submit the databank creation form. Once the process is completed, you will be redirected to the databank list window.</p>
</div>
<div class="section" id="editing-a-databank">
<h3>Editing a databank<a class="headerlink" href="#editing-a-databank" title="Permalink to this headline">¶</a></h3>
<p>You can edit all information concerning an existing databank except its annotation type, the sequence file used and whether it contains decoy sequences.
From the databank list window, click on the <code class="docutils literal notranslate"><span class="pre">Edit</span></code> button on the right side of the databank row. A form similar of that used to add a databank will be displayed. Make the desired changes and click on the <code class="docutils literal notranslate"><span class="pre">Save</span></code> button to validate your changes.
You can test your annotation rules as described above for databank addition but regardless of the databank file origine.</p>
<p>If your databank references a Mascot file, it is possible to check if the file has been updated on the mascot server by clicking on the <code class="docutils literal notranslate"><span class="pre">Check</span> <span class="pre">for</span> <span class="pre">update</span></code> button. This can take up to a few minutes for large databank files such as NCBI databanks. Checking for file update is not mandatory since it will be performed automatically once the databank is used during an Analysis import.</p>
</div>
<div class="section" id="deleting-a-databank">
<h3>Deleting a databank<a class="headerlink" href="#deleting-a-databank" title="Permalink to this headline">¶</a></h3>
<p>We recommend to delete any databank that will no longer be used to keep the list displayed as short as possible. Deletion of a databank has no effect on the traceability information of analyses using this databank. A databank can be deleted at any time except during import of analyses using this databank.</p>
<p>From the databank list window, click on the <code class="docutils literal notranslate"><span class="pre">Delete</span></code> button on the right side of the databank row. A prompt will asked you to confirm your decision.</p>
</div>
</div>
<div class="section" id="spectral-swath-libraries-management">
<h2>Spectral (SWATH) libraries management<a class="headerlink" href="#spectral-swath-libraries-management" title="Permalink to this headline">¶</a></h2>
<div class="section" id="listing-spectral-libraries">
<h3>Listing spectral libraries<a class="headerlink" href="#listing-spectral-libraries" title="Permalink to this headline">¶</a></h3>
<p>From <em>myProMS</em> main window, select <code class="docutils literal notranslate"><span class="pre">Annotation</span> <span class="pre">data</span></code> and follow the SWATH libraries link. All available libraries are listed in alphabetic order with few informations as shown in the screenshot below. On the left side are listed all existing libraries with the possibility to delete, export, edit or update them. On the upper part of the window, you can either add a new library, merge two existing ones, or visualize running processes. You can also search if some desired proteins are existing into one library thank to the search link and restore the previous version of an updated library.</p>
<img alt="../../_images/list_spectral_libraries.png" src="../../_images/list_spectral_libraries.png" />
</div>
<div class="section" id="adding-a-new-library">
<h3>Adding a new library<a class="headerlink" href="#adding-a-new-library" title="Permalink to this headline">¶</a></h3>
<p>From the libraries list window, click on the <code class="docutils literal notranslate"><span class="pre">Add</span> <span class="pre">new</span> <span class="pre">spectral</span> <span class="pre">library</span></code> button on the top of the list to display the form below.</p>
<img alt="../../_images/add_spectral_library.png" src="../../_images/add_spectral_library.png" />
<dl class="docutils">
<dt>You need to select the following parameters in the library creation form :</dt>
<dd><ul class="first last simple">
<li><strong>Task</strong> : You can create a new library or merge new data files with an existing library (create a new library from an existing one).</li>
<li><strong>Library name</strong> : Provide a name for the library.</li>
<li><strong>Species</strong> : Select the species scientific name to filter the databank list.</li>
<li><strong>Consensus library options</strong> : A consensus library is a spectral library in which MS2 spectrum entries with a redundant peptide sequence assignment have been collapsed into a single entry. Two options are provided for consensus library generation: a simple option that assumes that all fragment ion spectra are correctly assigned (UNSPLIT) and a more sophisticated option that additionally considers retention time when merging spectra (SPLIT).</li>
<li><strong>Files</strong> : Select the DDA data files used to generate the spectral library. Data from 3 search engines can be selected : Mascot files (.dat), X! Tandem files (.xml or .tandem.pep.xml) and Sequest (.xml). For each Mascot, X! Tandem or Sequest file you need to upload the associated mzXML file (with the same name as the Mascot, X! Tandem or Sequest file). You can upload your files from your computer, you can import them from an existing project (only for the .dat files), upload an archive, or select the files in the shared directory.</li>
<li><strong>Instrument</strong> : The mass spectrometer used to acquire the data.</li>
<li><strong>Databank</strong> : The fasta file used by the search engines (Mascot,  X! Tandem and Sequest).</li>
<li><strong>Mayu options</strong> : FDR estimation with MAYU. False Discovery Rate (FDR) and number of missed cleavage can be selected.</li>
<li><strong>RT file</strong> : The file containing the list of iRT retention time reference peptides.</li>
<li><strong>Description</strong> : Optional description of the current library.</li>
</ul>
</dd>
</dl>
<p>Once the form is filled, click on the <code class="docutils literal notranslate"><span class="pre">Submit</span></code> button to launch the spectral library creation process.</p>
<div class="admonition danger">
<p class="first admonition-title">Danger</p>
<p class="last">Update with the new version</p>
</div>
</div>
<div class="section" id="merging-two-library">
<h3>Merging two library<a class="headerlink" href="#merging-two-library" title="Permalink to this headline">¶</a></h3>
<p>Two libraries can be merged by clicking on the <code class="docutils literal notranslate"><span class="pre">Merge</span> <span class="pre">two</span> <span class="pre">libraries</span></code> button on the libraries list window. The displayed form requires the names of each of the 2 libraries, the name of the new library and an optional description. Clicking on Submit will fuse the selected libraries to create the new library.</p>
<div class="admonition important">
<p class="first admonition-title">Important</p>
<p class="last">Only two libraries with the same iRT file, databank type and consensus library option (SPLIT or UNSPLIT) can be merged.</p>
</div>
</div>
<div class="section" id="editing-a-library">
<h3>Editing a library<a class="headerlink" href="#editing-a-library" title="Permalink to this headline">¶</a></h3>
<p>From the libraries list window, click on the <code class="docutils literal notranslate"><span class="pre">Edit</span></code> button on the right side of the library row. The following form will be displayed :</p>
<img alt="../../_images/edit_spectral_library.png" src="../../_images/edit_spectral_library.png" />
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">Only the name and the description can be modified</p>
</div>
<p>Make the desired changes and click on the <code class="docutils literal notranslate"><span class="pre">Submit</span></code> button to save your changes.</p>
</div>
<div class="section" id="updating-a-library">
<h3>Updating a library<a class="headerlink" href="#updating-a-library" title="Permalink to this headline">¶</a></h3>
<p>It is also possible to extend a library using another databank-search data from the same organism. From the libraries list window, click on the Update button on the right of the library row. A form similar to the library creation one will be displayed.</p>
<p>Fill in the parameters and click on the <code class="docutils literal notranslate"><span class="pre">Submit</span></code> button to launch the update process.</p>
</div>
<div class="section" id="restoring-the-previous-version-of-a-library">
<h3>Restoring the previous version of a library<a class="headerlink" href="#restoring-the-previous-version-of-a-library" title="Permalink to this headline">¶</a></h3>
<p>An updated library can be downgraded by clicking on the <code class="docutils literal notranslate"><span class="pre">Restore</span> <span class="pre">previous</span> <span class="pre">version</span></code> button on the right of the library row on the libraries list interface. Every version of a library can be restored by consecutive downgrades.</p>
</div>
<div class="section" id="searching-for-proteins-in-a-library">
<h3>Searching for proteins in a library<a class="headerlink" href="#searching-for-proteins-in-a-library" title="Permalink to this headline">¶</a></h3>
<p>Another available option is to check whether a protein of interest is present in a library and visualize the associated peptides by clicking on the <code class="docutils literal notranslate"><span class="pre">Search</span></code> button of the desired library, on the libraries list window.</p>
<p>Several proteins can be searched at the same time by inserting the <strong>accession names</strong>, the <strong>protein id</strong> or the <strong>names</strong> of the proteins (one per line or separated by either comma or a space character) in the following form.</p>
<img alt="../../_images/search_protein_spectral_libraries.png" src="../../_images/search_protein_spectral_libraries.png" />
<p>All the selected terms are searched beforehand in <strong>Uniprot</strong>, and a list of proteins is displayed. Some information such as the protein name, id, accession number, length and corresponding gene names are shown. The number of associated peptides identified is also indicated.</p>
<img alt="../../_images/search_results_protein_spectral_libraries.png" src="../../_images/search_results_protein_spectral_libraries.png" />
<p>The peptide list and the protein’s sequence can be displayed by clicking on the number in the <code class="docutils literal notranslate"><span class="pre">#Peptides</span></code> column.</p>
<img alt="../../_images/protein_peptides_list.png" src="../../_images/protein_peptides_list.png" />
<p>Some information about each peptide such as sequence, modifications, position on the protein, M/Z, charge, IRT time and specificity are shown.</p>
</div>
<div class="section" id="exporting-a-library">
<h3>Exporting a library<a class="headerlink" href="#exporting-a-library" title="Permalink to this headline">¶</a></h3>
<p>You can export a library to use it in a quantification software. From the libraries list interface, click on the <code class="docutils literal notranslate"><span class="pre">Export</span></code> button to display the export form.</p>
<img alt="../../_images/export_spectral_library.png" src="../../_images/export_spectral_library.png" />
<dl class="docutils">
<dt>You have to fill in the following parameters :</dt>
<dd><ul class="first last simple">
<li><strong>Export format</strong> : The library can be exported for PeakView or for OpenSWATH, or you can download the final format of the library (sptxt).</li>
<li><strong>Mass range of fragment ions</strong> : Lower and upper mass limits of fragment ions. (min=350 and max=2000 by default).</li>
<li><strong>Ion series and charge</strong> : The ion desired type (a, b, c, x, y, or z) and charge separated by a comma. (charge=1+ and 2+ by default).</li>
<li><strong>Number of ions per peptide</strong> : Minimum and maximum number of ions per peptide. (min=3 and max=20 by default).</li>
<li><strong>Files</strong> :<ul>
<li><strong>Windows SWATH file</strong> : Upload the file that contain the SWATH window scheme that has been used for SWATH data acquisition.</li>
<li><strong>File with modifications delta mass</strong> : Optional file containing the modifications not specified by default.</li>
<li><strong>Labelling file</strong> : Optional file containing the amino acid isotopic labelling mass shifts. If this option is used, heavy transitions will be generated.</li>
<li><strong>Fasta file</strong> : Optional databank fasta file used to relate peptides to their proteins.</li>
</ul>
</li>
<li><strong>Other options</strong> : You can select another optional options such as the maximum permissible error, the time scale, the UIS order (calculated when using switching modification; if -1 is set, all transitions for each isoform will be reported; default : 2), or the list of allowed fragment mass modifications.</li>
<li><strong>Protein list</strong> : You can select a file containing a protein list to export just these proteins from the library.</li>
</ul>
</dd>
</dl>
<p>Then you can click on the <code class="docutils literal notranslate"><span class="pre">Submit</span></code> button to launch the export process. Once the process is complete, you can download the final file with a download link that will appear.</p>
</div>
<div class="section" id="deleting-a-spectral-library">
<h3>Deleting a spectral library<a class="headerlink" href="#deleting-a-spectral-library" title="Permalink to this headline">¶</a></h3>
<p>A library can be deleted from the list window (by clicking on the <code class="docutils literal notranslate"><span class="pre">Delete</span></code> button of the corresponding library) only if this library was not used to create another library (merge option, in that case, a prompt will inform you).</p>
</div>
</div>
<div class="section" id="go-files-management">
<h2>GO files management<a class="headerlink" href="#go-files-management" title="Permalink to this headline">¶</a></h2>
<p>GO analyses require two types of GO files: an ontology file and an annotation file. These files are not project-specific and are thus managed globally in <em>myProMS</em>.
From <em>myProMS</em> main window, select <code class="docutils literal notranslate"><span class="pre">Annotation</span> <span class="pre">data</span></code> and follow the <code class="docutils literal notranslate"><span class="pre">GO</span> <span class="pre">annotations</span></code> link to display the list of GO files recorded.</p>
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">Only bioinformaticians and authorized massists/managers can manage GO files</p>
</div>
<div class="section" id="ontology-files">
<h3>Ontology files<a class="headerlink" href="#ontology-files" title="Permalink to this headline">¶</a></h3>
<p>Ontology files contain the GO terms identifiers, description and relationships between. To add a new ontology file, click on <code class="docutils literal notranslate"><span class="pre">Add</span> <span class="pre">new</span> <span class="pre">Gene</span> <span class="pre">Ontology</span> <span class="pre">file</span></code>:</p>
<img alt="../../_images/add_go_file.jpg" src="../../_images/add_go_file.jpg" />
<dl class="docutils">
<dt>The displayed form requires the following information:</dt>
<dd><ul class="first last">
<li><p class="first"><strong>Name</strong>: A relevant name for the ontology. This name will be displayed in all GO analysis starting forms in ontology selecting section.</p>
</li>
<li><p class="first"><strong>File</strong>: The file containing the ontology must be in OBO format (not XML nor database dump). Daily updated ontology files can be fetched from GO website. The file can be uploaded directly from user computer or directly retrieved from remote FTP by writing its full URL (e.g. <a class="reference external" href="ftp://ftp.geneontology.org/pub/go/ontology/obo_format_1_2/gene_ontology.1_2.obo">ftp://ftp.geneontology.org/pub/go/ontology/obo_format_1_2/gene_ontology.1_2.obo</a>).</p>
</li>
<li><p class="first"><strong>Scope</strong>: Specify if the ontology file contains the full gene ontology or a slim version.</p>
<blockquote>
<div><div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">A slim version gives a broad overview of the ontology content without the detail of the specific fine grained terms.
If a slim file is used, make sure to select the slim option.</p>
</div>
<p>To be able to use a slim ontology for GO analyses, at least one full ontology file must have been also recorded to allows <em>myProMS</em> to reconstructs missing associations between proteins and the GO terms recorded in the slim file.</p>
<div class="admonition warning">
<p class="first admonition-title">Warning</p>
<p class="last">Running a slim GO analysis without a corresponding full ontology will cause an error</p>
</div>
</div></blockquote>
</li>
</ul>
</dd>
</dl>
<p>Saved ontologies can be edited. If the file was retrieved by FTP and a most recent version available on the distant server, it can be downloaded again directly by clicking on <code class="docutils literal notranslate"><span class="pre">Update</span> <span class="pre">file</span></code>.</p>
</div>
<div class="section" id="annotation-files">
<h3>Annotation files<a class="headerlink" href="#annotation-files" title="Permalink to this headline">¶</a></h3>
<p>Annotation files contains mapping of protein identifiers to GO terms. They are species-specific and must be in Gene Association File (GAF) format. A large number of updated annotation files for many species can be fetched from the Uniprot-GOA database. To add a new annotation, click on <code class="docutils literal notranslate"><span class="pre">Add</span> <span class="pre">new</span> <span class="pre">annotation</span> <span class="pre">file</span></code>:</p>
<img alt="../../_images/add_annotation_file.jpg" src="../../_images/add_annotation_file.jpg" />
<dl class="docutils">
<dt>The displayed form requires the following information:</dt>
<dd><ul class="first last">
<li><p class="first"><strong>Name</strong>: A relevant name for the annotation, that will be displayed on each GO analysis starting form in annotation selection section.</p>
</li>
<li><p class="first"><strong>Description</strong>: An optional description for the annotation.</p>
</li>
<li><p class="first"><strong>Species</strong>: Select the targeted species from the list of available ones (See Species below for more information).</p>
</li>
<li><p class="first"><strong>File</strong>: file can be uploaded from your computer or retrieved remotely from a FTP server (e.g. <a class="reference external" href="ftp://ftp.ebi.ac.uk/pub/databases/GO/goa/HUMAN/gene_association.goa_human.gz">ftp://ftp.ebi.ac.uk/pub/databases/GO/goa/HUMAN/gene_association.goa_human.gz</a> for the human annotation file).</p>
</li>
<li><p class="first"><strong>Identifier used</strong>: Select the protein identifier that must be used in <em>myProMS</em> to match the annotation’s one (eg: select Uniprot ID or Uniprot AC for Uniprot-GOA files). If “Default” is selected, the default protein identifier displayed in <em>myProMS</em> will be used.</p>
<blockquote>
<div><div class="admonition warning">
<p class="first admonition-title">Warning</p>
<p class="last"><strong>Identifier</strong> must be set carefully to insure proper GO annotation mapping</p>
</div>
</div></blockquote>
</li>
</ul>
</dd>
</dl>
</div>
</div>
<div class="section" id="species">
<h2>Species<a class="headerlink" href="#species" title="Permalink to this headline">¶</a></h2>
<p><em>myProMS</em> automatically records the species associated with any protein validated. Because different strains or variants of the same species are also recorded, it is necessary to manually link these entries to the <strong>same reference</strong> species. Furthermore, reference species must be recorded for Gene Ontology analyses. A species management section is provided so that bioinformaticians and authorized massists/managers can manually record or correct species information. By default, a list of 5 model organisms species data is provided with <em>myProMS</em> as reference.</p>
<div class="section" id="listing-species">
<h3>Listing species<a class="headerlink" href="#listing-species" title="Permalink to this headline">¶</a></h3>
<p>From <em>myProMS</em> main window, select Annotation data and follow the <code class="docutils literal notranslate"><span class="pre">Species</span></code> link to access the species management interface.</p>
<p>&lt;Figure Species list&gt;</p>
<p>As shown is the above screen capture, a subset of species can be listed either by <strong>scientific</strong> or <strong>common name</strong> by selecting the appropriate initial letter in one of the 2 alphabets displayed.</p>
</div>
<div class="section" id="adding-or-editing-a-species">
<h3>Adding or editing a species<a class="headerlink" href="#adding-or-editing-a-species" title="Permalink to this headline">¶</a></h3>
<p>A species can be added or edited by clicking on “Add species“ or “Edit” buttons respectively. The following form is then displayed:</p>
<p>&lt;Figure Add/edit species&gt;</p>
<p>The common name, scientific name and taxonID fields are mandatory. A link to the <strong>NCBI Taxonomy</strong> resource is provided to help you find this information if not known. You can either set this species as reference by checking the <code class="docutils literal notranslate"><span class="pre">Is</span> <span class="pre">reference</span></code> or link it to a reference one In addition an optional field allows you to link any species with a reference one by selecting a target species in the drop-down menu.</p>
</div>
<div class="section" id="deleting-a-species">
<h3>Deleting a species<a class="headerlink" href="#deleting-a-species" title="Permalink to this headline">¶</a></h3>
<p>A species can be deleted from the list interface (by clicking on the <code class="docutils literal notranslate"><span class="pre">Delete</span></code> button of the corresponding species) only if this species is no longer associated with validated protein, not set as reference species nor used in a GO analysis.</p>
</div>
<div class="section" id="sequence-modifications">
<h3>Sequence modifications<a class="headerlink" href="#sequence-modifications" title="Permalink to this headline">¶</a></h3>
<p><em>myProMS</em> automatically records the post-translational modifications (PTMs) found in imported analyses on protein sequences. Once an analysis has been imported, PTMs found on this analysis are added to the list and you can edit the properties of those PTMs. <em>myProMS</em> keeps track of every imported modifications and displays by ascending name as defined on UNIMOD website.</p>
<p>In this list, all PTMs are depicted by five informations ; names (PSI-MS and interim name), description, specificity and status (red or green). The specificity describes on which residue the PTM tends to occur. It could be on a specific residue including or not a context (like “Any N-term”, “Protein N-term”,etc.).</p>
<img alt="../../_images/list_modifications.png" src="../../_images/list_modifications.png" />
<p>The status displayed on the upper left corner by a circle tells if a PTM is valid or not. To be valid, a PTM should be characterized by a monoisotopic and an average mass like the ones defined on UNIMOD website. If a PTM is not valid (<img alt="ptm_not_valid" src="../../_images/ptm_not_valid.gif" />), it means that <em>myProMS</em> could not retrieved this PTM through the <a class="reference external" href="http://www.unimod.org/unimod_help.html">UNIMOD</a> current list of PTMs. The origin of that issue comes from one reason : you entered a “home-named” modification that is not referenced in UNIMOD.</p>
<dl class="docutils">
<dt>Two solutions exist to solve this issue :</dt>
<dd><ul class="first last simple">
<li>If this modification was already imported on another referenced name, you should merge this “home-named” modification to this one by editing the non-valid PTM. In the future, <em>myProMS</em> will automatically applies to this “home-named” modification the properties of the referenced one.</li>
<li>If this modification was not imported through another name, you should edit the PTM and provide mass and specificity.</li>
</ul>
</dd>
</dl>
<p>Make sure that all PTMs retrieved are valid in order to avoid the other features available in <em>myProMS</em> to give wrong output (like fragmentation table of peptides for example).</p>
</div>
</div>
<div class="section" id="editing-or-merging-ptms">
<h2>Editing or merging PTMs<a class="headerlink" href="#editing-or-merging-ptms" title="Permalink to this headline">¶</a></h2>
<p>A PTM can be edited by clicking on <code class="docutils literal notranslate"><span class="pre">Edit</span></code> button.</p>
<img alt="../../_images/edit_modifications.png" src="../../_images/edit_modifications.png" />
<p>In this mode, you can update the description or the delta-mass of this PTM. A link to UNIMOD is provided by giving the Unimod Accession number. Specificity can be updated given your expertise on the PTM and reviews articles you may have read.</p>
<p>The option <code class="docutils literal notranslate"><span class="pre">Merge</span> <span class="pre">with</span></code> gives the opportunity to merge two PTMs into one single entry. This could be useful if you wish to give an alternative name to a modification. Select the modification you want to merge with the current PTM and click on <code class="docutils literal notranslate"><span class="pre">Save</span></code>. This action will add the name of the current modification to the list of alternative names of the one selected.</p>
<p>For PTMs that you want to make appear in your projects and give special attention to, you need to enter a code (usually, a single letter) and a color. Those PTMs will become relevant and will be selectable in every project you manage.</p>
<p>Here is a list of relevant PTMs and their associated code-color designation:</p>
<img alt="../../_images/list_relevant_modifications.png" src="../../_images/list_relevant_modifications.png" />
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">For more information on that topic, please, see Project <a class="reference internal" href="../project_management/project_settings.html#project-creation"><span class="std std-ref">Creation</span></a>.</p>
</div>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="server_administration.html" class="btn btn-neutral float-right" title="Server administration" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="users_management.html" class="btn btn-neutral float-left" title="Users management" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2019, Patrick Poullet, Valentin Sabatet, Victor Laigle

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>