

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Analysis import &mdash; myProMS  documentation</title>
  

  
  
  
  

  
  <script type="text/javascript" src="../../_static/js/modernizr.min.js"></script>
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
        <script type="text/javascript" src="../../_static/jquery.js"></script>
        <script type="text/javascript" src="../../_static/underscore.js"></script>
        <script type="text/javascript" src="../../_static/doctools.js"></script>
        <script type="text/javascript" src="../../_static/language_data.js"></script>
    
    <script type="text/javascript" src="../../_static/js/theme.js"></script>

    

  
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Analysis validation" href="analysis_validation.html" />
    <link rel="prev" title="Projects settings" href="project_settings.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../../index.html" class="icon icon-home"> myProMS
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Getting Started</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/installation.html">Installation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../getting_started/installation.html#git-repository">Git Repository</a></li>
<li class="toctree-l2"><a class="reference internal" href="../getting_started/installation.html#docker-deployment">Docker deployment</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/login.html">Login</a></li>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/server_architecture.html">Server Architecture</a></li>
</ul>
<p class="caption"><span class="caption-text">Server Management</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../server_management/users_management.html">Users management</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../server_management/users_management.html#user-classes-and-access-privileges">User classes and access privileges</a></li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/users_management.html#account-management">Account management</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Project management</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="project_settings.html">Projects settings</a><ul>
<li class="toctree-l2"><a class="reference internal" href="project_settings.html#selection">Selection</a></li>
<li class="toctree-l2"><a class="reference internal" href="project_settings.html#creation">Creation</a></li>
<li class="toctree-l2"><a class="reference internal" href="project_settings.html#accessibility">Accessibility</a></li>
<li class="toctree-l2"><a class="reference internal" href="project_settings.html#organization">Organization</a><ul>
<li class="toctree-l3"><a class="reference internal" href="project_settings.html#experiments"><strong>Experiments</strong></a></li>
<li class="toctree-l3"><a class="reference internal" href="project_settings.html#samples"><strong>Samples</strong></a></li>
<li class="toctree-l3"><a class="reference internal" href="project_settings.html#analyses"><strong>Analyses</strong></a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="project_settings.html#navigation">Navigation</a></li>
<li class="toctree-l2"><a class="reference internal" href="project_settings.html#life-span">Life span</a></li>
</ul>
</li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Analysis import</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#supported-search-engines">Supported search engines</a></li>
<li class="toctree-l2"><a class="reference internal" href="#collecting-search-files">Collecting search files</a></li>
<li class="toctree-l2"><a class="reference internal" href="#import-parameters-mascot-proteome-discoverer-and-x-tandem">Import parameters (Mascot, Proteome Discoverer and X!Tandem)</a></li>
<li class="toctree-l2"><a class="reference internal" href="#importing-maxquant-data">Importing MaxQuant data</a></li>
<li class="toctree-l2"><a class="reference internal" href="#import-dia-data">Import DIA data</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#from-peakview">From PeakView</a></li>
<li class="toctree-l3"><a class="reference internal" href="#from-openswath">From OpenSWATH</a></li>
<li class="toctree-l3"><a class="reference internal" href="#running-openswath-quantification">Running OpenSWATH quantification</a></li>
<li class="toctree-l3"><a class="reference internal" href="#from-spectronaut">From Spectronaut</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#analysis-summary">Analysis summary</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="analysis_validation.html">Analysis validation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="analysis_validation.html#automated-peptide-protein-validation">Automated peptide/protein validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="analysis_validation.html#fdr-false-discovery-rate-based-validation">FDR (False discovery rate) - based validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="analysis_validation.html#qualitative-validation">Qualitative validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="analysis_validation.html#comparative-validation">Comparative validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="analysis_validation.html#validation-templates">Validation templates</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="analysis_validation.html#manual-peptide-protein-validation">Manual peptide/protein validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="analysis_validation.html#peptides-selection-exclusion">Peptides selection/exclusion</a></li>
<li class="toctree-l3"><a class="reference internal" href="analysis_validation.html#protein-exclusion-and-filtering">Protein exclusion and filtering</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="analysis_validation.html#lower-scoring-peptides-activation">Lower-scoring peptides activation</a></li>
<li class="toctree-l2"><a class="reference internal" href="analysis_validation.html#clear-peptide-protein-selections">Clear peptide/protein selections</a></li>
<li class="toctree-l2"><a class="reference internal" href="analysis_validation.html#sequence-modification-validation">Sequence modification validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="analysis_validation.html#phosphorylation-sites-validation-with-phosphors">Phosphorylation sites validation with PhosphoRS</a></li>
<li class="toctree-l3"><a class="reference internal" href="analysis_validation.html#manual-validation-of-modifications">Manual validation of modifications</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="analysis_validation.html#validation-traceability">Validation traceability</a></li>
<li class="toctree-l2"><a class="reference internal" href="analysis_validation.html#reporting">Reporting</a></li>
</ul>
</li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">myProMS</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../index.html">Docs</a> &raquo;</li>
        
      <li>Analysis import</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="../../_sources/docs/project_management/analysis_import.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="analysis-import">
<span id="id1"></span><h1>Analysis import<a class="headerlink" href="#analysis-import" title="Permalink to this headline">¶</a></h1>
<p>The collection of spectra/peptides/proteins(/quantification) data contained in a search result file are imported into myProMS as an <strong>Analysis</strong>.</p>
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">Only bioinformaticians, massists and managers can import Analyses.</p>
</div>
<p>Select the <strong>Experiment</strong>, <strong>Sample</strong> or <strong>2D Gel</strong> into which the Analyses must be imported and click <code class="docutils literal notranslate"><span class="pre">Process</span> <span class="pre">Analyses</span></code> in the option frame. From the selection menu displayed, select <code class="docutils literal notranslate"><span class="pre">Analysis</span> <span class="pre">Management</span></code> to display the list of available options.</p>
<img alt="../../_images/process_import_analysis.png" src="../../_images/process_import_analysis.png" />
<div class="section" id="supported-search-engines">
<h2>Supported search engines<a class="headerlink" href="#supported-search-engines" title="Permalink to this headline">¶</a></h2>
<dl class="docutils">
<dt>myProMS allows to import from various search engines :</dt>
<dd><ul class="first last simple">
<li>Mascot (DAT file or MSF file from Proteome Discoverer Software by Thermo Scientific).</li>
<li>Paragon (XML generated from ProteinPilotTM Software by AB SCIEX, group2xml.exe).</li>
<li>Sequest (MSF file from Proteome Discoverer Software by Thermo Scientific).</li>
<li>Phenyx (XML file generated through Phenyx platform by GeneBio). DEPRECATED!</li>
<li>Andromeda/MaxQuant (mqpar.xml and 3-4 txt files are required).</li>
<li>X!Tandem (XML file from X! Tandem pipeline (<a class="reference external" href="http://pappso.inra.fr/bioinfo/xtandempipeline/download.php">PAPPSO</a>) or from Trans-Proteomic Pipeline (<a class="reference external" href="http://tools.proteomecenter.org/wiki/index.php?title=Software:TPP">TPP</a> )).</li>
<li>PeakView (Exported Excel XLXS file for SWATH data).</li>
<li>OpenSwath (TSV file from <a class="reference external" href="http://www.openswath.org/en/latest/">OpenSWATH</a> workflow).</li>
<li>Spectronaut (TSV file generated from SpectraunotTM by Biognosys).</li>
</ul>
</dd>
</dl>
<div class="admonition attention">
<p class="first admonition-title">Attention</p>
<p class="last">If you perform Mascot searches with Proteome Discoverer (PD) Software, make sure you dit not use Protein and Peptide filters.
In the <strong>Workflow Editor</strong>, click on Mascot node and then, set the four filters Peptide Cutt Off Score, <cite>Peptide Without Protein Cut</cite>, <cite>Protein Relevance Threshold</cite> and <cite>Protein Relevance Factor</cite> to 0.
If these filters are not turned-off, myProMS import options such as predefined <cite>False Discovery Rate (FDR)</cite> will not be accurate.</p>
</div>
<img alt="../../_images/workflow_editor.png" src="../../_images/workflow_editor.png" />
</div>
<div class="section" id="collecting-search-files">
<h2>Collecting search files<a class="headerlink" href="#collecting-search-files" title="Permalink to this headline">¶</a></h2>
<p>Multiple analyses can be imported at once as long as the corresponding searches were performed with the same search engine and protein databank(s).</p>
<p>To select data files in <strong>Mascot</strong>, <strong>Proteome Discoverer</strong> and <strong>X!Tandem</strong>:</p>
<p>Click on <code class="docutils literal notranslate"><span class="pre">Proceed</span></code> next to the <code class="docutils literal notranslate"><span class="pre">Import</span> <span class="pre">multiple</span> <span class="pre">analyses</span></code> process as shown below. The following form will be displayed to select the source of the search files to be imported.</p>
<dl class="docutils">
<dt>Multiple import sources are available:</dt>
<dd><ul class="first last simple">
<li><strong>A user directory on server</strong>: Following upload, files are stored in a user-dedicated directory on server. These files will stay available to the user until he decides to delete them; either just after import or later. In the later case the user can still access this directory for file management purpose by clicking on the <code class="docutils literal notranslate"><span class="pre">Clean</span> <span class="pre">My</span> <span class="pre">Directory</span></code> button.</li>
<li><strong>Any directory on server</strong>: This option is available to bioinformaticians only. The user can provide any path on the server where myProMS should look for search results files.</li>
<li><strong>Mascot server</strong>: If a Mascot server is declared in myProMS configuration file, it can be accessed, searched for specific search results and files directly uploaded into myProMS user directory.</li>
</ul>
</dd>
</dl>
<p>User can search results files by <em>date</em> or <em>job number</em> range or keywords in the files <em>search title</em>.
The list matching files is then displayed and grouped by day of creation.
Specific information on a file (name, availability, search title and user ID) can be displayed by clicking on the file name.
If access to Mascot is restricted (user accounts set up), the Mascot userID must be defined in myProMS as well (see <a class="reference internal" href="../server_management/users_management.html#account-management"><span class="std std-ref">Account management</span></a>).</p>
<dl class="docutils">
<dt>In this case, only Mascot files accessible to the user will be displayed.</dt>
<dd><ul class="first last simple">
<li><strong>Upload Zip archive</strong>: If a large number of files must be imported, they can be uploaded at once as the zip archive. The archive will be unzipped on the server.</li>
<li><strong>Upload multiple files</strong>: Alternatively, up to 10 files can be uploaded as separate files.</li>
</ul>
</dd>
</dl>
<p>Once your files have been selected, click on <code class="docutils literal notranslate"><span class="pre">Proceed</span></code> to initiate file retrieval from the selected source. This procedure may take a few minutes depending of the number and size of the files. Once the transfer is complete, a file import interface will be displayed.</p>
<div class="admonition important">
<p class="first admonition-title">Important</p>
<p class="last">Most browsers do not support upload of files with (total) size &gt; <strong>2 Gb</strong>.
If files larger than 2 Gb must be uploaded, we recommend to use <strong>Google Chrome</strong>.
This limitation does not apply when retrieving files directly from a Mascot server.</p>
</div>
</div>
<div class="section" id="import-parameters-mascot-proteome-discoverer-and-x-tandem">
<h2>Import parameters (Mascot, Proteome Discoverer and X!Tandem)<a class="headerlink" href="#import-parameters-mascot-proteome-discoverer-and-x-tandem" title="Permalink to this headline">¶</a></h2>
<p>Files retrieved are listed in alphabetical order together with pertinent information about the search performed: the MS file, search type (MS2, MS1 or mix of both), databank(s) and taxonomy and search title used.</p>
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">The same Proteome Discoverer msf file can contain multiple searches results (e.g. a search performed with Mascot and another with Sequest). In this case, separate entries will be listed for each search performed together with more details on the parameters used. Each search result can be imported separately and distinct Analysis items will be created.</p>
</div>
<dl class="docutils">
<dt>Proceed as follow to continue data import:</dt>
<dd><ol class="first last arabic">
<li><p class="first">Select the files to be imported from the list by checking the boxes on the left-hand side of the files name.</p>
</li>
<li><p class="first">Provide a name for the Analysis to be created (Analysis name column). This name can be typed by the user or set to match either the name of the search file or that of the original MS file used for the search.</p>
</li>
<li><p class="first">If  the parent item was an Experiment (or a 2D Gel), each new Analysis must be associated with a specific Sample (or Spot). Pre-existing Samples (or Spots) can be selected from a dropdown menu or the can be created on the fly (Samples only): To create a new Sample, select “New” from the dropdown menu of Parents column. A popup window will ask you to provide a name for the new Sample.</p>
</li>
<li><p class="first">Select the databanks to be used to extract protein annotations (Databank(s) field). If multiple databanks were used during the search (possible with Mascot for instance), the corresponding number should also be selected here. All search files to be imported in the same batch should have been performed using the same or equivalent (set of) databank(s).</p>
</li>
<li><dl class="first docutils">
<dt>Define a filtering rule for the data to be imported (<strong>Threshold score</strong>):</dt>
<dd><ul class="first last">
<li><p class="first">If a decoy search was performed, data can be filtered to based on a user-defined <strong>False Discovery Rate (FDR)</strong> on peptide identification (default is 1%). A threshold score for peptide identification will be determined so that the data imported will (tentatively) match the defined FDR value as illustrated in the figure below. Threshold score calculation can either uses the qvality algorithm <a class="footnote-reference" href="#id3" id="id2">[1]</a>, the <strong>Mayu</strong> algorithm or the DT count algorithm. In <strong>DT count</strong> mode, decoy (D) and target (T) peptides are simultaneously counted in descending score order until the proportion of the 2 populations matches the selected FDR value.</p>
<blockquote>
<div><img alt="../../_images/analysis_import_FDR.png" src="../../_images/analysis_import_FDR.png" />
</div></blockquote>
</li>
<li><p class="first">If no decoy search was performed or the FDR value was set equal or less than 0, the filtering will be performed according to a minimum (<strong>threshold</strong>) score for peptide identification. A default (search engine-specific) threshold score will be applied unless a different one is provided by the user.</p>
</li>
</ul>
</dd>
</dl>
</li>
<li><p class="first">Select the maximum number of interpretations allowed of the same fragmentation spectrum (Max. rank). The default is 1, but up to 10 can be chosen.</p>
<blockquote>
<div><div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">When performing FDR filtering, it is recommended to set this value to 1 since the FDR calculation is based on 1 interpretation per spectrum.</p>
</div>
</div></blockquote>
</li>
<li><p class="first">Provide optional <strong>description</strong> and <strong>comments</strong>.</p>
</li>
<li><p class="first">Decide whether the files should be deleted after import or not (<code class="docutils literal notranslate"><span class="pre">Delete</span> <span class="pre">imported</span> <span class="pre">files</span> <span class="pre">afterwards</span></code>). Unless selected for deletion, file will remain on the server for new import until the user decides to delete them.</p>
</li>
<li><p class="first">Click on <code class="docutils literal notranslate"><span class="pre">Proceed</span></code> to initiate the data import into myProMS database.</p>
</li>
</ol>
</dd>
</dl>
</div>
<div class="section" id="importing-maxquant-data">
<h2>Importing MaxQuant data<a class="headerlink" href="#importing-maxquant-data" title="Permalink to this headline">¶</a></h2>
<dl class="docutils">
<dt>3 to 5 files are required to import a MaxQuant search/quantification into myProMS:</dt>
<dd><ol class="first last arabic simple">
<li>mqpar.xml (usually located in the root directory of the MaxQuant search),</li>
<li>evidence.txt (file from the Combined/txt directory),</li>
<li>Peptides.txt (idem),</li>
<li>proteinGroups.txt (idem. Optional, only to import protein quantification data),</li>
<li>msms.txt (idem. Optional, only to display peptide fragmentation spectra).</li>
</ol>
</dd>
</dl>
<p>Files 2 to 5 must be compressed in a common archive before import.
Select an Experiment in which to import the data.
From the Process Analyses window, select either the <code class="docutils literal notranslate"><span class="pre">Import</span> <span class="pre">multiple</span> <span class="pre">Analyses</span></code> or <code class="docutils literal notranslate"><span class="pre">Import</span> <span class="pre">quantification</span></code> processes and click on <code class="docutils literal notranslate"><span class="pre">Proceed</span></code> next to <code class="docutils literal notranslate"><span class="pre">Import</span> <span class="pre">MaxQuant</span> <span class="pre">quantification</span></code> to display the form below.</p>
<p>&lt;Image&gt;</p>
<p>Provide the files mentioned above and the protein sequence databank(s) used for the search. If contaminants were searched, provide a matching contaminant databank. Finally, select the rule you wish to use for protein aggregation into match groups: myProMS or MaxQuant. Specifiy also if you wish to import protein quantification data (the proteinGroups.txt file must be provided in the archive in that case). Submit the form to start data import. Data import will take a few minutes. Samples, Analyses and an experimental Design, a peptide quantification and 1 to multiple protein quantifications will be added to the selected Experiment according to the information extracted from the files uploaded. Peptides and proteins will be automatically validated since they were used in protein quantification.</p>
</div>
<div class="section" id="import-dia-data">
<h2>Import DIA data<a class="headerlink" href="#import-dia-data" title="Permalink to this headline">¶</a></h2>
<p>User can import DIA quantification data from three different software : <strong>PeakView</strong>, <strong>OpenSWATH</strong> and <strong>Spectronaut</strong>.
Into myProMS select an Experiment and from the <code class="docutils literal notranslate"><span class="pre">Process</span> <span class="pre">Analyses</span></code> window, select the <code class="docutils literal notranslate"><span class="pre">Analysis</span> <span class="pre">quantification</span></code> process and click on <code class="docutils literal notranslate"><span class="pre">Proceed</span></code> next to <code class="docutils literal notranslate"><span class="pre">Import</span> <span class="pre">PeakView/OpenSWATH/Spectronaut</span></code> data to display the associated form.</p>
<div class="section" id="from-peakview">
<h3>From PeakView<a class="headerlink" href="#from-peakview" title="Permalink to this headline">¶</a></h3>
<p>Two files are required : the <cite>Excel worksheet</cite> file generated by PeakView and the <cite>spectral library</cite>.
Into PeakView, once the experimental SWATH data analysis is over, you can export the result into an Excel file by clicking on the <code class="docutils literal notranslate"><span class="pre">Quantitation</span></code> tab on the toolbar and selecting <code class="docutils literal notranslate"><span class="pre">SWATH</span> <span class="pre">Processing/Export/All</span></code>.
The PeakView search parameters can be filled in the following form to be saved in myProMS database to ensure traceability. Then submit the form to launch data import.</p>
<img alt="../../_images/analysis_import_DIA_PeakView.png" src="../../_images/analysis_import_DIA_PeakView.png" />
</div>
<div class="section" id="from-openswath">
<h3>From OpenSWATH<a class="headerlink" href="#from-openswath" title="Permalink to this headline">¶</a></h3>
<p>The file from OpenSWATH is required, this is a TSV file generated by the last step of the workflow (TRIC). The library that will be used to analyse the experimental data is also needed as the associated export parameter file. Then, the user must provide the TRIC method used and the number version of OpenSWATH.</p>
<img alt="../../_images/analysis_import_DIA_OpenSwath.png" src="../../_images/analysis_import_DIA_OpenSwath.png" />
</div>
<div class="section" id="running-openswath-quantification">
<h3>Running OpenSWATH quantification<a class="headerlink" href="#running-openswath-quantification" title="Permalink to this headline">¶</a></h3>
<p>The OpenSWATH workflow can be launched directly from myProMS. The process will analyse the experimental files and import the result in a same step. Some parameters are required as the library name, the mzXML results files, the iRT file (in TraML format) and the DIA windows file.  The user can import his own library converted for OpenSWATH, and choose to merge this result with other existing analysis.</p>
<img alt="../../_images/analysis_import_DIA_OpenSwath_Quanti.png" src="../../_images/analysis_import_DIA_OpenSwath_Quanti.png" />
<div class="admonition danger">
<p class="first admonition-title">Danger</p>
<p class="last">A MODIFIER AVEC LE NOUVEAU WORKFLOW</p>
</div>
</div>
<div class="section" id="from-spectronaut">
<h3>From Spectronaut<a class="headerlink" href="#from-spectronaut" title="Permalink to this headline">¶</a></h3>
<p>The import Spectronaut data form is similar to OpenSwath’s form.</p>
<img alt="../../_images/analysis_import_DIA_Spectronaut.png" src="../../_images/analysis_import_DIA_Spectronaut.png" />
<p>Once the process is over, samples and analyses will be added to the selected Experiment.
Peptides and proteins will be automatically validated.
Transition quantification data will be imported but not the peptide or the protein quantification results as a dedicated pipeline is available in myProMS to perform this task (see the Protein quantification chapter below for more information).</p>
</div>
</div>
<div class="section" id="analysis-summary">
<h2>Analysis summary<a class="headerlink" href="#analysis-summary" title="Permalink to this headline">¶</a></h2>
<p>Files will be imported sequentially and the Analyses (and Samples if any) newly generated will appear in the top left navigation window.
The Analysis summary is shown below:</p>
<img alt="../../_images/analysis_import_summary.png" src="../../_images/analysis_import_summary.png" />
<p>Diverse information is available to the user such as the <cite>MS type</cite>, <cite>search engine</cite>, <cite>databank(s)</cite> (selected in myProMS), <cite>protein identifier type</cite>, <cite>taxonomy</cite>, <cite>labeling method</cite> if any, <cite>threshold score value and strategy</cite> (FDR-based or user-defined), <cite>validation status</cite>,…
More search parameters and score distribution for FDR computation can be displayed on demand.</p>
<table class="docutils footnote" frame="void" id="id3" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label"><a class="fn-backref" href="#id2">[1]</a></td><td>Qvality algorithm : <a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/19193729">Käll et al. Bioinformatics 2008, 25</a></td></tr>
</tbody>
</table>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="analysis_validation.html" class="btn btn-neutral float-right" title="Analysis validation" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="project_settings.html" class="btn btn-neutral float-left" title="Projects settings" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2019, Patrick Poullet, Valentin Sabatet, Victor Laigle

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>