

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Peptide Quantification &mdash; myProMS  documentation</title>
  

  
  
  
  

  
  <script type="text/javascript" src="../../_static/js/modernizr.min.js"></script>
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
        <script type="text/javascript" src="../../_static/jquery.js"></script>
        <script type="text/javascript" src="../../_static/underscore.js"></script>
        <script type="text/javascript" src="../../_static/doctools.js"></script>
        <script type="text/javascript" src="../../_static/language_data.js"></script>
    
    <script type="text/javascript" src="../../_static/js/theme.js"></script>

    

  
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/css/custom.css" type="text/css" />
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="next" title="Protein Quantification" href="protein_quantification.html" />
    <link rel="prev" title="Compare project proteins" href="../protein_identification/compare_proteins.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../../index.html" class="icon icon-home"> myProMS
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Getting Started</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/installation.html">Installation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../getting_started/installation.html#git-repository">Git Repository</a></li>
<li class="toctree-l2"><a class="reference internal" href="../getting_started/installation.html#docker-deployment">Docker deployment</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/login.html">Login</a></li>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/server_architecture.html">Server Architecture</a></li>
</ul>
<p class="caption"><span class="caption-text">Server Management</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../server_management/users_management.html">Users management</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../server_management/users_management.html#user-classes-and-access-privileges">User classes and access privileges</a></li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/users_management.html#account-management">Account management</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../server_management/annotation_data.html">Annotation data management</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../server_management/annotation_data.html#sequence-databanks">Sequence databanks</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#databank-types">Databank types</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#listing-databanks">Listing databanks</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#adding-a-new-databank">Adding a new databank</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#editing-a-databank">Editing a databank</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#deleting-a-databank">Deleting a databank</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/annotation_data.html#spectral-swath-libraries-management">Spectral (SWATH) libraries management</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#listing-spectral-libraries">Listing spectral libraries</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#adding-a-new-library">Adding a new library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#merging-two-library">Merging two library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#editing-a-library">Editing a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#updating-a-library">Updating a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#restoring-the-previous-version-of-a-library">Restoring the previous version of a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#searching-for-proteins-in-a-library">Searching for proteins in a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#exporting-a-library">Exporting a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#deleting-a-spectral-library">Deleting a spectral library</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/annotation_data.html#go-files-management">GO files management</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#ontology-files">Ontology files</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#annotation-files">Annotation files</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/annotation_data.html#species">Species</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#listing-species">Listing species</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#adding-or-editing-a-species">Adding or editing a species</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#deleting-a-species">Deleting a species</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#sequence-modifications">Sequence modifications</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/annotation_data.html#editing-or-merging-ptms">Editing or merging PTMs</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../server_management/server_administration.html">Server administration</a></li>
</ul>
<p class="caption"><span class="caption-text">Project management</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../project_management/project_settings.html">Projects settings</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#selection">Selection</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#creation">Creation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#accessibility">Accessibility</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#organization">Organization</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/project_settings.html#experiments"><strong>Experiments</strong></a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/project_settings.html#samples"><strong>Samples</strong></a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/project_settings.html#analyses"><strong>Analyses</strong></a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#navigation">Navigation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#life-span">Life span</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../project_management/analysis_import.html">Analysis import</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#supported-search-engines">Supported search engines</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#collecting-search-files">Collecting search files</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#import-parameters-mascot-proteome-discoverer-and-x-tandem">Import parameters (Mascot, Proteome Discoverer and X!Tandem)</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#importing-maxquant-data">Importing MaxQuant data</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#import-dia-data">Import DIA data</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#from-peakview">From PeakView</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#from-openswath">From OpenSWATH</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#running-openswath-quantification">Running OpenSWATH quantification</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#from-spectronaut">From Spectronaut</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#analysis-summary">Analysis summary</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../project_management/analysis_validation.html">Analysis validation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#automated-peptide-protein-validation">Automated peptide/protein validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#fdr-false-discovery-rate-based-validation">FDR (False discovery rate) - based validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#qualitative-validation">Qualitative validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#comparative-validation">Comparative validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#validation-templates">Validation templates</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#manual-peptide-protein-validation">Manual peptide/protein validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#peptides-selection-exclusion">Peptides selection/exclusion</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#protein-exclusion-and-filtering">Protein exclusion and filtering</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#lower-scoring-peptides-activation">Lower-scoring peptides activation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#clear-peptide-protein-selections">Clear peptide/protein selections</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#sequence-modification-validation">Sequence modification validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#phosphorylation-sites-validation-with-phosphors">Phosphorylation sites validation with PhosphoRS</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#manual-validation-of-modifications">Manual validation of modifications</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#validation-traceability">Validation traceability</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#reporting">Reporting</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../project_management/biological_samples.html">Biological Samples</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#properties">Properties</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#treatments">Treatments</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#recording-a-biological-sample">Recording a biological sample</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#linking-biological-samples-to-ms-analyses">Linking biological samples to MS Analyses</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Protein Identification</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/validated_proteins.html">Validated proteins</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/validated_proteins.html#match-groups-and-protein-visibility">Match groups and protein visibility</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#top-protein-selection-rules">Top protein selection rules</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#project-wide-protein-visibility-rules">Project-wide protein visibility rules</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#checking-for-conflicting-match-groups">Checking for conflicting match groups</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#displaying-match-group-composition">Displaying match group composition</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#manual-edition">Manual edition</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#peptide-distribution-in-match-group">Peptide distribution in match group</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/validated_proteins.html#identifier-mapping">Identifier mapping</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/validated_proteins.html#single-protein-view">Single protein view</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/protein_lists.html">Protein lists</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/protein_lists.html#project-based-protein-lists">Project-based protein lists</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/protein_lists.html#user-defined-protein-lists">User-defined protein lists</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/protein_lists.html#export-protein-lists">Export protein lists</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/search_proteins.html">Search for proteins</a></li>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/compare_proteins.html">Compare project proteins</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#full-protein-level-comparison">Full protein-level comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#pairwise-protein-level-comparison">Pairwise protein-level comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#pairwise-peptide-level-comparison">Pairwise peptide-level comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#saving-a-comparison">Saving a comparison</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Quantification</span></p>
<ul class="current">
<li class="toctree-l1 current"><a class="current reference internal" href="#">Peptide Quantification</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#data-import-from-search-results-file">Data import from search results file</a></li>
<li class="toctree-l2"><a class="reference internal" href="#data-extraction-from-lc-ms-file-with-masschroq">Data extraction from LC/MS file with MassChroQ</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#managing-mzxml-files">Managing mzXML files</a></li>
<li class="toctree-l3"><a class="reference internal" href="#running-xic-extraction">Running XIC extraction</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#virtual-ghost-peptides-and-proteins">Virtual/Ghost peptides and proteins</a></li>
<li class="toctree-l2"><a class="reference internal" href="#displaying-peptide-quantification-data">Displaying peptide quantification data</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="protein_quantification.html">Protein Quantification</a><ul>
<li class="toctree-l2"><a class="reference internal" href="protein_quantification.html#absolute-abundance-quantification">Absolute abundance quantification</a><ul>
<li class="toctree-l3"><a class="reference internal" href="protein_quantification.html#empai-label-free">emPAI (label-free)</a></li>
<li class="toctree-l3"><a class="reference internal" href="protein_quantification.html#sin-label-free">SIn (label-free)</a></li>
<li class="toctree-l3"><a class="reference internal" href="protein_quantification.html#maxquant-intensity-lfq-ibaq">MaxQuant: Intensity, LFQ, iBAQ</a></li>
<li class="toctree-l3"><a class="reference internal" href="protein_quantification.html#displaying-single-abundance-quantification-data">Displaying single abundance quantification data</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="protein_quantification.html#relative-abundance-quantification">Relative abundance quantification</a><ul>
<li class="toctree-l3"><a class="reference internal" href="protein_quantification.html#single-analysis-quantification-labeled">Single-Analysis quantification (labeled)</a></li>
<li class="toctree-l3"><a class="reference internal" href="protein_quantification.html#design-based-quantifications">Design-based quantifications</a></li>
<li class="toctree-l3"><a class="reference internal" href="protein_quantification.html#displaying-relative-abundance-quantification-data">Displaying relative abundance quantification data</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="protein_quantification.html#label-free-quantifications">Label-free quantifications</a><ul>
<li class="toctree-l3"><a class="reference internal" href="protein_quantification.html#tnpq">TnPQ</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="protein_quantification.html#comparing-multiple-protein-quantifications">Comparing multiple protein quantifications</a></li>
<li class="toctree-l2"><a class="reference internal" href="protein_quantification.html#exporting-multiple-quantifications">Exporting multiple quantifications</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="ptms_quantification.html">PTMs quantification</a><ul>
<li class="toctree-l2"><a class="reference internal" href="ptms_quantification.html#set-ptms-relevance-to-project">Set PTMs relevance to project</a></li>
<li class="toctree-l2"><a class="reference internal" href="ptms_quantification.html#display-modification-sites-distribution">Display modification sites distribution</a></li>
<li class="toctree-l2"><a class="reference internal" href="ptms_quantification.html#compare-ptms-between-projects">Compare PTMs between projects</a></li>
<li class="toctree-l2"><a class="reference internal" href="ptms_quantification.html#quantify-modification-sites">Quantify modification sites</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Data Analysis</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../data_analysis/exploratory_analysis.html">Exploratory analysis</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#launching-exploratory-analyses-on-protein-quantification">Launching exploratory analyses on protein quantification</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#launching-exploratory-analyses-on-peptides-count">Launching exploratory analyses on peptides count</a></li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#principal-component-analysis-pca">Principal Component Analysis (PCA)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#summary-edit-delete">Summary / edit / delete</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#displaying-a-pca">Displaying a PCA</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#d-clustering">2D-Clustering</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#id1">Summary / edit / delete</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/exploratory_analysis.html#displaying-clustering">Displaying clustering</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../data_analysis/functional_analysis.html">Functional analysis</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/functional_analysis.html#gene-ontology">Gene Ontology</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#go-summary">GO summary</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#go-enrichment-analysis">GO enrichment analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#quantitative-gene-enrichment-analysis">Quantitative gene enrichment analysis</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/functional_analysis.html#pathway-enrichment">Pathway enrichment</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#launch-pathway-analysis">Launch pathway analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#summary-edit-delete">Summary / edit / delete</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#displaying-pathway-analysis">Displaying pathway analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#export">Export</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../data_analysis/functional_analysis.html#motif-enrichment-analysis">Motif enrichment analysis</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#launch-analysis">Launch analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#summary-editing-deleting">Summary/editing/deleting</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#displaying-motif-enrichment">Displaying motif enrichment</a></li>
<li class="toctree-l3"><a class="reference internal" href="../data_analysis/functional_analysis.html#heatmap-motif-analysis">Heatmap motif analysis</a></li>
</ul>
</li>
</ul>
</li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">myProMS</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../index.html">Docs</a> &raquo;</li>
        
      <li>Peptide Quantification</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="../../_sources/docs/quantification/peptide_quantification.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="peptide-quantification">
<span id="id1"></span><h1>Peptide Quantification<a class="headerlink" href="#peptide-quantification" title="Permalink to this headline">¶</a></h1>
<p>Peptide quantification is a necessary step for peptide-based protein quantification; whether the quantification is based on MS-spectra (SILAC, TnPQ, XIC-based label-free quantification,…) or on MS/MS fragments (iTRAQ, TMT, SWATH…).</p>
<div class="section" id="data-import-from-search-results-file">
<h2>Data import from search results file<a class="headerlink" href="#data-import-from-search-results-file" title="Permalink to this headline">¶</a></h2>
<p>Some search result files already contain peptide quantification data.
It is always the case for MS/MS fragment-based quantification such as iTRAQ for which the peptide ion intensities are part of the MS/MS spectrum data.</p>
<p>Some search results files (<em>Proteome Discoverer MSF</em> or <em>MaxQuant</em>) may also contain peptide XIC data if a quantification was performed after the search process.
When peptide quantification data are contained in the imported search results file(s) myProMS will automatically import these data either during search data import or at the <code class="docutils literal notranslate"><span class="pre">Validated</span> <span class="pre">data</span> <span class="pre">Report</span></code> step if data validation must be performed.</p>
<p>Only quantification data related to validated peptide will be kept (see the Virtual peptides section below for important additional information).</p>
</div>
<div class="section" id="data-extraction-from-lc-ms-file-with-masschroq">
<h2>Data extraction from LC/MS file with MassChroQ<a class="headerlink" href="#data-extraction-from-lc-ms-file-with-masschroq" title="Permalink to this headline">¶</a></h2>
<p>XIC-based peptide quantification can be performed within myProMS whether or not peptide quantification data were already available in the search results file. myProMS uses the tool <strong>MassChroQ</strong> <a class="footnote-reference" href="#id3" id="id2">[1]</a> to perform this task. However, the corresponding LC/MS file(s) must be provided in <strong>mzXML</strong> format.</p>
<div class="section" id="managing-mzxml-files">
<h3>Managing mzXML files<a class="headerlink" href="#managing-mzxml-files" title="Permalink to this headline">¶</a></h3>
<p>To manage the list of mzXML files available within a given project, select any <em>Experiment</em> or <em>Sample</em> from the project navigation window and click on the <code class="docutils literal notranslate"><span class="pre">Process</span> <span class="pre">Analyses</span></code> button in the option frame. From the selection menu displayed, select <code class="docutils literal notranslate"><span class="pre">Analysis</span> <span class="pre">Quantification</span></code> to display the list of available options. Click on <code class="docutils literal notranslate"><span class="pre">Proceed</span> <span class="pre">next</span></code> to the Manage mzXML files process as shown below.</p>
<img alt="../../_images/process_quantification.png" src="../../_images/process_quantification.png" />
<p>The following form will be displayed to allow you to either <em>import</em> a new mzXML file or <em>delete</em> already imported ones.</p>
<img alt="../../_images/manage_mzxml.png" src="../../_images/manage_mzxml.png" />
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">We perform LC/MS files (RAW &amp; WIFF formats) conversion to mzXML with <a class="reference external" href="http://proteowizard.sourceforge.net/">ProteoWizard</a> tool using default settings. Other format conversion tools were not tested. We also recommend not to change the files name (except for the mzXML extension) to ease Analysis/mzXLM file matching in the quantification launch step.</p>
</div>
</div>
<div class="section" id="running-xic-extraction">
<h3>Running XIC extraction<a class="headerlink" href="#running-xic-extraction" title="Permalink to this headline">¶</a></h3>
<p>Go to the <code class="docutils literal notranslate"><span class="pre">Analysis</span> <span class="pre">Quantification</span></code> options (as shown above) and click on <code class="docutils literal notranslate"><span class="pre">Proceed</span></code> next to the <strong>XIC extraction with MassChroQ</strong> process to display the form shown below.</p>
<img alt="../../_images/run_masschroq.png" src="../../_images/run_masschroq.png" />
<dl class="docutils">
<dt>In the first part of the form, multiple parameters can be set for the extraction:</dt>
<dd><ul class="first last">
<li><p class="first"><strong>Name of the quantification</strong> : All extraction data collected will be regrouped in a single quantification carrying this name.</p>
</li>
<li><p class="first"><strong>Extraction type</strong> : Profile or centroid</p>
</li>
<li><p class="first"><strong>Isotope labeling</strong> : If isotope labeling was performed on your sample, it is possible to use XIC extraction to retrieve it. To do so, you need to choose <code class="docutils literal notranslate"><span class="pre">SILAC</span></code>. Up to 3 different channels can be retrieved at a time (e.g. <em>heavy</em>, <em>light</em> and <em>medium</em>) that have to be named. For each channel, one or more quantification label can be added given the experimental design. Each quantification label is linked to a post-translational modification that explains it. Specify the modification target on which it occurs (<em>side chain</em>, <em>n-ter</em> or <em>c-ter</em>). If side chain is chosen, don’t forget to give the residue where the post-translational modification occurs.</p>
<blockquote>
<div><p>Here is provided an example of the filled form in a SILAC experiment where lysine and arginine residues were designed as heavy isotope. 13C6-15N4 was renamed to Arg10 for clarity.</p>
<div class="admonition important">
<p class="first admonition-title">Important</p>
<p class="last">It is really important at this stage to define a light channel if a biological experiment/condition/analysis match the light version. Otherwise, the light version of the peptide will not be retrieved in the end.</p>
</div>
<img alt="../../_images/run_silac.jpg" src="../../_images/run_silac.jpg" />
<p><br></p>
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">Your modification is not selectable in the modification target option ? Check the status of the modification (see Sequence modification section below). Maybe the modification you are using is not tagged as label. If not, change this and save it by editing the modification.</p>
</div>
</div></blockquote>
</li>
<li><p class="first">Alignment settings: Multiple LC/MS runs can be quantified at once. MassChroQ can align all runs to match features across different runs. User must provide an alignment algorithm (OBI-Warp or ms2), a reference run by selecting the corresponding analysis and an m/z window (for OBI-Warp algorithm).</p>
</li>
<li><p class="first">Peptide selection: Whether to extract or not all charge states of a given peptide.</p>
</li>
<li><p class="first">Type of XIC extraction to be performed: basePeak area (most intense peak in the range of masses) or TIC area (summed intensity across the range of masses).</p>
</li>
<li><p class="first">More settings are also available by clicking on the corresponding button.</p>
</li>
</ul>
</dd>
</dl>
<p>Finally, click on <code class="docutils literal notranslate"><span class="pre">Launch</span> <span class="pre">Quantification</span></code> to start the extraction. A pop-up window will appear to allow you to monitor the quantification progress. XIC extraction is a long process that can last up to an hour or more depending on the number of Analyses to be aligned, the complexity of the LC/MS run and the computer power available. You can continue using myProMS in the mean time and even launch other quantifications. All on-going quantification jobs are displayed in the <code class="docutils literal notranslate"><span class="pre">Monitor</span> <span class="pre">Quantifications</span></code> window (see figure below).</p>
<p>&lt;Figure Monitor Quantification window&gt;</p>
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">As new jobs are launched or old ones completed, they will appear or disappear from the list. Additionally, if an error occurs during quantification, a message will appear for the corresponding job. The user will be able to display the content of the error message and delete the failed job and all associated temporary data.</p>
</div>
<p>If this window is closed inadvertently (closing it has no effects on the on-going jobs) or did not appear (pop-up windows for myProMS URL must be enabled in your browser), it can be displayed again by clicking on the <code class="docutils literal notranslate"><span class="pre">Monitor</span> <span class="pre">on-going</span> <span class="pre">quantifications</span></code> button in the <code class="docutils literal notranslate"><span class="pre">Analysis</span> <span class="pre">Quantification</span></code> options (<code class="docutils literal notranslate"><span class="pre">Process</span> <span class="pre">Analyses</span></code> &gt; <code class="docutils literal notranslate"><span class="pre">Analysis</span> <span class="pre">Quantification</span></code>).</p>
<p>In the second part of the form, you must select the analyses corresponding to the extraction and associate its mzXML file to each of them.
If the mzXML file name matches the MS data file recorded for the Analysis, myProMS will do the job for you.
(Check the <a class="reference external" href="http://pappso.inra.fr/bioinfo/masschroq/manual/index.html">MassChroQ manual</a> for help on setting these parameters properly).</p>
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">Only runs with reproducible retention-time values (e.g biological or technical replicates) should be selected for alignment. Runs potential very different set of feature (e.g.sample  fractions separated on a gel) should be extracted separately.</p>
</div>
</div>
</div>
<div class="section" id="virtual-ghost-peptides-and-proteins">
<h2>Virtual/Ghost peptides and proteins<a class="headerlink" href="#virtual-ghost-peptides-and-proteins" title="Permalink to this headline">¶</a></h2>
<p>During the quantification process, intensities of parent ions can be calculated even though the corresponding peptide did not end up in the list of peptides validated. For instance, in the case of a SILAC-labeled analysis, the label-free form of a peptide can be validated but not its labeled counterpart; either because the later falls under the threshold score used or was not identified at all.
However, these data are valuable for the quantification since both peptide forms are required for ratio calculation.</p>
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last"><em>myProMS</em> solves this issue by adding these missing peptides to the list of validated peptides but with a special status: <strong>virtual peptides</strong> (also called <strong>ghost peptide</strong>).</p>
</div>
<p>This strategy of peptide addition also applies to alternative charge states of a given peptides (e.g. if the 2+ charge state of a peptide is validated, all other quantified charge states will be added as virtual peptides). Virtual peptides remain hidden unless their presence is required for proper data interpretation.</p>
<div class="admonition important">
<p class="first admonition-title">Important</p>
<p class="last">When applied to a label-free quantification where 2 or more analyses are aligned, a peptide validated in the reference analysis but missing (or not validated) in an aligned one can be reextracted as a <strong>virtual peptide</strong>. If this peptide does not belong to any validated proteins of the aligned analysis, the protein(s) matching this peptide in the reference will be added to the analysis aligned as <strong>ghost protein(s)</strong>. Ghost proteins appear in italics in most protein lists.</p>
</div>
</div>
<div class="section" id="displaying-peptide-quantification-data">
<h2>Displaying peptide quantification data<a class="headerlink" href="#displaying-peptide-quantification-data" title="Permalink to this headline">¶</a></h2>
<p>Once the peptide quantification data are available (after a <strong>Report</strong> for Search file extraction or an <strong>XIC extraction</strong> within myProMS), they can be displayed for individual analysis by selecting the corresponding analysis in the <code class="docutils literal notranslate"><span class="pre">Project</span> <span class="pre">navigation</span></code> frame and clicking on the <code class="docutils literal notranslate"><span class="pre">Internal</span> <span class="pre">Quantifications</span></code> button in the option frame.</p>
<p>From new window displayed in the result frame, select the name of the quantification (in the <code class="docutils literal notranslate"><span class="pre">Peptide</span> <span class="pre">quantification</span></code> section). A window similar to the one below will be displayed showing a summary of the quantification parameters used (if any: no parameters are displayed in case of a direct extraction from a search results file) and a list of proteins with identified peptides and corresponding XICs or fragments area for DIA extraction. In case of labeled quantification, peptide sets (label isoforms) are grouped into a single peptide row. The peptide set sequence, variable modification, position, charge, score(s) and XIC(s) are displayed.</p>
<div class="figure" id="id4">
<img alt="SILAC Quantification results" src="../../_images/silac_quanti_results.png" />
<p class="caption"><span class="caption-text"><em>Legend: Case of a direct extraction of SILAC-labeled peptide XIC from a search result file</em></span></p>
</div>
<p>Virtual peptides can be easily identified as they do not have score.</p>
<div class="figure" id="id5">
<img alt="XIC multi-results" src="../../_images/xic_multi_results.png" />
<p class="caption"><span class="caption-text"><em>Legend: Case of a MassChroQ extraction with alignment of multiple Analyses</em></span></p>
</div>
<table class="docutils footnote" frame="void" id="id3" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label"><a class="fn-backref" href="#id2">[1]</a></td><td>MassChroQ : <a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/21751374">Valot B et al, Proteomics, 2011</a></td></tr>
</tbody>
</table>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
        <a href="protein_quantification.html" class="btn btn-neutral float-right" title="Protein Quantification" accesskey="n" rel="next">Next <span class="fa fa-arrow-circle-right"></span></a>
      
      
        <a href="../protein_identification/compare_proteins.html" class="btn btn-neutral float-left" title="Compare project proteins" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2019, Patrick Poullet, Valentin Sabatet, Victor Laigle

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>