

<!DOCTYPE html>
<!--[if IE 8]><html class="no-js lt-ie9" lang="en" > <![endif]-->
<!--[if gt IE 8]><!--> <html class="no-js" lang="en" > <!--<![endif]-->
<head>
  <meta charset="utf-8">
  
  <meta name="viewport" content="width=device-width, initial-scale=1.0">
  
  <title>Functional analysis &mdash; myProMS  documentation</title>
  

  
  
  
  

  
  <script type="text/javascript" src="../../_static/js/modernizr.min.js"></script>
  
    
      <script type="text/javascript" id="documentation_options" data-url_root="../../" src="../../_static/documentation_options.js"></script>
        <script type="text/javascript" src="../../_static/jquery.js"></script>
        <script type="text/javascript" src="../../_static/underscore.js"></script>
        <script type="text/javascript" src="../../_static/doctools.js"></script>
        <script type="text/javascript" src="../../_static/language_data.js"></script>
    
    <script type="text/javascript" src="../../_static/js/theme.js"></script>

    

  
  <link rel="stylesheet" href="../../_static/css/theme.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/pygments.css" type="text/css" />
  <link rel="stylesheet" href="../../_static/css/custom.css" type="text/css" />
    <link rel="index" title="Index" href="../../genindex.html" />
    <link rel="search" title="Search" href="../../search.html" />
    <link rel="prev" title="Exploratory analysis" href="exploratory_analysis.html" /> 
</head>

<body class="wy-body-for-nav">

   
  <div class="wy-grid-for-nav">
    
    <nav data-toggle="wy-nav-shift" class="wy-nav-side">
      <div class="wy-side-scroll">
        <div class="wy-side-nav-search" >
          

          
            <a href="../../index.html" class="icon icon-home"> myProMS
          

          
          </a>

          
            
            
          

          
<div role="search">
  <form id="rtd-search-form" class="wy-form" action="../../search.html" method="get">
    <input type="text" name="q" placeholder="Search docs" />
    <input type="hidden" name="check_keywords" value="yes" />
    <input type="hidden" name="area" value="default" />
  </form>
</div>

          
        </div>

        <div class="wy-menu wy-menu-vertical" data-spy="affix" role="navigation" aria-label="main navigation">
          
            
            
              
            
            
              <p class="caption"><span class="caption-text">Getting Started</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/installation.html">Installation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../getting_started/installation.html#git-repository">Git Repository</a></li>
<li class="toctree-l2"><a class="reference internal" href="../getting_started/installation.html#docker-deployment">Docker deployment</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/login.html">Login</a></li>
<li class="toctree-l1"><a class="reference internal" href="../getting_started/server_architecture.html">Server Architecture</a></li>
</ul>
<p class="caption"><span class="caption-text">Server Management</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../server_management/users_management.html">Users management</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../server_management/users_management.html#user-classes-and-access-privileges">User classes and access privileges</a></li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/users_management.html#account-management">Account management</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../server_management/annotation_data.html">Annotation data management</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../server_management/annotation_data.html#sequence-databanks">Sequence databanks</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#databank-types">Databank types</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#listing-databanks">Listing databanks</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#adding-a-new-databank">Adding a new databank</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#editing-a-databank">Editing a databank</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#deleting-a-databank">Deleting a databank</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/annotation_data.html#spectral-swath-libraries-management">Spectral (SWATH) libraries management</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#listing-spectral-libraries">Listing spectral libraries</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#adding-a-new-library">Adding a new library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#merging-two-library">Merging two library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#editing-a-library">Editing a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#updating-a-library">Updating a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#restoring-the-previous-version-of-a-library">Restoring the previous version of a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#searching-for-proteins-in-a-library">Searching for proteins in a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#exporting-a-library">Exporting a library</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#deleting-a-spectral-library">Deleting a spectral library</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/annotation_data.html#go-files-management">GO files management</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#ontology-files">Ontology files</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#annotation-files">Annotation files</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/annotation_data.html#species">Species</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#listing-species">Listing species</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#adding-or-editing-a-species">Adding or editing a species</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#deleting-a-species">Deleting a species</a></li>
<li class="toctree-l3"><a class="reference internal" href="../server_management/annotation_data.html#sequence-modifications">Sequence modifications</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../server_management/annotation_data.html#editing-or-merging-ptms">Editing or merging PTMs</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../server_management/server_administration.html">Server administration</a></li>
</ul>
<p class="caption"><span class="caption-text">Project management</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../project_management/project_settings.html">Projects settings</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#selection">Selection</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#creation">Creation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#accessibility">Accessibility</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#organization">Organization</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/project_settings.html#experiments"><strong>Experiments</strong></a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/project_settings.html#samples"><strong>Samples</strong></a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/project_settings.html#analyses"><strong>Analyses</strong></a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#navigation">Navigation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/project_settings.html#life-span">Life span</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../project_management/analysis_import.html">Analysis import</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#supported-search-engines">Supported search engines</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#collecting-search-files">Collecting search files</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#import-parameters-mascot-proteome-discoverer-and-x-tandem">Import parameters (Mascot, Proteome Discoverer and X!Tandem)</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#importing-maxquant-data">Importing MaxQuant data</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#import-dia-data">Import DIA data</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#from-peakview">From PeakView</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#from-openswath">From OpenSWATH</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#running-openswath-quantification">Running OpenSWATH quantification</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_import.html#from-spectronaut">From Spectronaut</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_import.html#analysis-summary">Analysis summary</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../project_management/analysis_validation.html">Analysis validation</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#automated-peptide-protein-validation">Automated peptide/protein validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#fdr-false-discovery-rate-based-validation">FDR (False discovery rate) - based validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#qualitative-validation">Qualitative validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#comparative-validation">Comparative validation</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#validation-templates">Validation templates</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#manual-peptide-protein-validation">Manual peptide/protein validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#peptides-selection-exclusion">Peptides selection/exclusion</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#protein-exclusion-and-filtering">Protein exclusion and filtering</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#lower-scoring-peptides-activation">Lower-scoring peptides activation</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#clear-peptide-protein-selections">Clear peptide/protein selections</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#sequence-modification-validation">Sequence modification validation</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#phosphorylation-sites-validation-with-phosphors">Phosphorylation sites validation with PhosphoRS</a></li>
<li class="toctree-l3"><a class="reference internal" href="../project_management/analysis_validation.html#manual-validation-of-modifications">Manual validation of modifications</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#validation-traceability">Validation traceability</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/analysis_validation.html#reporting">Reporting</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../project_management/biological_samples.html">Biological Samples</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#properties">Properties</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#treatments">Treatments</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#recording-a-biological-sample">Recording a biological sample</a></li>
<li class="toctree-l2"><a class="reference internal" href="../project_management/biological_samples.html#linking-biological-samples-to-ms-analyses">Linking biological samples to MS Analyses</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Protein Identification</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/validated_proteins.html">Validated proteins</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/validated_proteins.html#match-groups-and-protein-visibility">Match groups and protein visibility</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#top-protein-selection-rules">Top protein selection rules</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#project-wide-protein-visibility-rules">Project-wide protein visibility rules</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#checking-for-conflicting-match-groups">Checking for conflicting match groups</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#displaying-match-group-composition">Displaying match group composition</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#manual-edition">Manual edition</a></li>
<li class="toctree-l3"><a class="reference internal" href="../protein_identification/validated_proteins.html#peptide-distribution-in-match-group">Peptide distribution in match group</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/validated_proteins.html#identifier-mapping">Identifier mapping</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/validated_proteins.html#single-protein-view">Single protein view</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/protein_lists.html">Protein lists</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/protein_lists.html#project-based-protein-lists">Project-based protein lists</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/protein_lists.html#user-defined-protein-lists">User-defined protein lists</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/protein_lists.html#export-protein-lists">Export protein lists</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/search_proteins.html">Search for proteins</a></li>
<li class="toctree-l1"><a class="reference internal" href="../protein_identification/compare_proteins.html">Compare project proteins</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#full-protein-level-comparison">Full protein-level comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#pairwise-protein-level-comparison">Pairwise protein-level comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#pairwise-peptide-level-comparison">Pairwise peptide-level comparison</a></li>
<li class="toctree-l2"><a class="reference internal" href="../protein_identification/compare_proteins.html#saving-a-comparison">Saving a comparison</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Quantification</span></p>
<ul>
<li class="toctree-l1"><a class="reference internal" href="../quantification/peptide_quantification.html">Peptide Quantification</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../quantification/peptide_quantification.html#data-import-from-search-results-file">Data import from search results file</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/peptide_quantification.html#data-extraction-from-lc-ms-file-with-masschroq">Data extraction from LC/MS file with MassChroQ</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../quantification/peptide_quantification.html#managing-mzxml-files">Managing mzXML files</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/peptide_quantification.html#running-xic-extraction">Running XIC extraction</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/peptide_quantification.html#virtual-ghost-peptides-and-proteins">Virtual/Ghost peptides and proteins</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/peptide_quantification.html#displaying-peptide-quantification-data">Displaying peptide quantification data</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../quantification/protein_quantification.html">Protein Quantification</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../quantification/protein_quantification.html#absolute-abundance-quantification">Absolute abundance quantification</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#empai-label-free">emPAI (label-free)</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#sin-label-free">SIn (label-free)</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#maxquant-intensity-lfq-ibaq">MaxQuant: Intensity, LFQ, iBAQ</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#displaying-single-abundance-quantification-data">Displaying single abundance quantification data</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/protein_quantification.html#relative-abundance-quantification">Relative abundance quantification</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#single-analysis-quantification-labeled">Single-Analysis quantification (labeled)</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#design-based-quantifications">Design-based quantifications</a></li>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#displaying-relative-abundance-quantification-data">Displaying relative abundance quantification data</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/protein_quantification.html#label-free-quantifications">Label-free quantifications</a><ul>
<li class="toctree-l3"><a class="reference internal" href="../quantification/protein_quantification.html#tnpq">TnPQ</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/protein_quantification.html#comparing-multiple-protein-quantifications">Comparing multiple protein quantifications</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/protein_quantification.html#exporting-multiple-quantifications">Exporting multiple quantifications</a></li>
</ul>
</li>
<li class="toctree-l1"><a class="reference internal" href="../quantification/ptms_quantification.html">PTMs quantification</a><ul>
<li class="toctree-l2"><a class="reference internal" href="../quantification/ptms_quantification.html#set-ptms-relevance-to-project">Set PTMs relevance to project</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/ptms_quantification.html#display-modification-sites-distribution">Display modification sites distribution</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/ptms_quantification.html#compare-ptms-between-projects">Compare PTMs between projects</a></li>
<li class="toctree-l2"><a class="reference internal" href="../quantification/ptms_quantification.html#quantify-modification-sites">Quantify modification sites</a></li>
</ul>
</li>
</ul>
<p class="caption"><span class="caption-text">Data Analysis</span></p>
<ul class="current">
<li class="toctree-l1"><a class="reference internal" href="exploratory_analysis.html">Exploratory analysis</a><ul>
<li class="toctree-l2"><a class="reference internal" href="exploratory_analysis.html#launching-exploratory-analyses-on-protein-quantification">Launching exploratory analyses on protein quantification</a></li>
<li class="toctree-l2"><a class="reference internal" href="exploratory_analysis.html#launching-exploratory-analyses-on-peptides-count">Launching exploratory analyses on peptides count</a></li>
<li class="toctree-l2"><a class="reference internal" href="exploratory_analysis.html#principal-component-analysis-pca">Principal Component Analysis (PCA)</a><ul>
<li class="toctree-l3"><a class="reference internal" href="exploratory_analysis.html#summary-edit-delete">Summary / edit / delete</a></li>
<li class="toctree-l3"><a class="reference internal" href="exploratory_analysis.html#displaying-a-pca">Displaying a PCA</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="exploratory_analysis.html#d-clustering">2D-Clustering</a><ul>
<li class="toctree-l3"><a class="reference internal" href="exploratory_analysis.html#id1">Summary / edit / delete</a></li>
<li class="toctree-l3"><a class="reference internal" href="exploratory_analysis.html#displaying-clustering">Displaying clustering</a></li>
</ul>
</li>
</ul>
</li>
<li class="toctree-l1 current"><a class="current reference internal" href="#">Functional analysis</a><ul>
<li class="toctree-l2"><a class="reference internal" href="#gene-ontology">Gene Ontology</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#go-summary">GO summary</a></li>
<li class="toctree-l3"><a class="reference internal" href="#go-enrichment-analysis">GO enrichment analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="#quantitative-gene-enrichment-analysis">Quantitative gene enrichment analysis</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#pathway-enrichment">Pathway enrichment</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#launch-pathway-analysis">Launch pathway analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="#summary-edit-delete">Summary / edit / delete</a></li>
<li class="toctree-l3"><a class="reference internal" href="#displaying-pathway-analysis">Displaying pathway analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="#export">Export</a></li>
</ul>
</li>
<li class="toctree-l2"><a class="reference internal" href="#motif-enrichment-analysis">Motif enrichment analysis</a><ul>
<li class="toctree-l3"><a class="reference internal" href="#launch-analysis">Launch analysis</a></li>
<li class="toctree-l3"><a class="reference internal" href="#summary-editing-deleting">Summary/editing/deleting</a></li>
<li class="toctree-l3"><a class="reference internal" href="#displaying-motif-enrichment">Displaying motif enrichment</a></li>
<li class="toctree-l3"><a class="reference internal" href="#heatmap-motif-analysis">Heatmap motif analysis</a></li>
</ul>
</li>
</ul>
</li>
</ul>

            
          
        </div>
      </div>
    </nav>

    <section data-toggle="wy-nav-shift" class="wy-nav-content-wrap">

      
      <nav class="wy-nav-top" aria-label="top navigation">
        
          <i data-toggle="wy-nav-top" class="fa fa-bars"></i>
          <a href="../../index.html">myProMS</a>
        
      </nav>


      <div class="wy-nav-content">
        
        <div class="rst-content">
        
          















<div role="navigation" aria-label="breadcrumbs navigation">

  <ul class="wy-breadcrumbs">
    
      <li><a href="../../index.html">Docs</a> &raquo;</li>
        
      <li>Functional analysis</li>
    
    
      <li class="wy-breadcrumbs-aside">
        
            
            <a href="../../_sources/docs/data_analysis/functional_analysis.rst.txt" rel="nofollow"> View page source</a>
          
        
      </li>
    
  </ul>

  
  <hr/>
</div>
          <div role="main" class="document" itemscope="itemscope" itemtype="http://schema.org/Article">
           <div itemprop="articleBody">
            
  <div class="section" id="functional-analysis">
<h1>Functional analysis<a class="headerlink" href="#functional-analysis" title="Permalink to this headline">¶</a></h1>
<div class="section" id="gene-ontology">
<h2>Gene Ontology<a class="headerlink" href="#gene-ontology" title="Permalink to this headline">¶</a></h2>
<p>Different types of analyses using <strong>Gene Ontology (GO)</strong> can be performed on validated protein lists.
The GO project provides a controlled vocabulary of terms for describing gene products such as proteins.
For more details, see the <a class="reference external" href="http://www.geneontology.org/">GO website</a>.
A GO analysis can regroup proteins into standardized categories of terms belonging to 3 domains: <strong>Biological Process</strong>, <strong>Cellular Component</strong> and <strong>Molecular Function</strong>.</p>
<p>In myProMS, all GO analyses need 2 types of GO files that are managed from GO files management section (See corresponding chapter below for more information) :</p>
<ul class="simple">
<li><strong>Ontology file</strong>: the file that contains all term descriptions and their relationships between each other</li>
<li><strong>Annotation file</strong>: the file that maps each protein identifier to the most specific terms that characterize the protein.</li>
</ul>
<div class="section" id="go-summary">
<h3>GO summary<a class="headerlink" href="#go-summary" title="Permalink to this headline">¶</a></h3>
<p>The GO summary tool can be used to simply regroup proteins sharing common GO terms.
This tool can be run from the option frame on any project item, by clicking on the <code class="docutils literal notranslate"><span class="pre">Gene</span> <span class="pre">Ontology</span> <span class="pre">summary</span></code> button. The following form is then displayed:</p>
<img alt="../../_images/go_summary.jpg" src="../../_images/go_summary.jpg" />
<table border="1" class="docutils">
<colgroup>
<col width="11%" />
<col width="89%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td>Ontology</td>
<td>The file containing terms that will be used to regroup proteins</td>
</tr>
<tr class="row-even"><td>Annotation</td>
<td>The file containing protein annotations to GO terms</td>
</tr>
<tr class="row-odd"><td>Domain</td>
<td>Select one of the 3 GO domains the analysis will be focused on</td>
</tr>
<tr class="row-even"><td>Depth</td>
<td>Only terms at the specified depth in the GO graph structure will be used. Depth is calculated by counting the distance between a term and the root term of the corresponding ontology domain. If a high depth is selected, a very large number of terms will be displayed and the results may be difficult to read</td>
</tr>
<tr class="row-odd"><td>Minimal protein per term (Optional)</td>
<td>If a selected term contains less proteins that this specified value, this term will be ignored and the matching proteins will be added to the “Other” category</td>
</tr>
</tbody>
</table>
<p>Click on <code class="docutils literal notranslate"><span class="pre">Display</span></code> to launch the process. After a short calculation time, results are displayed as shown in the example below:</p>
<img alt="../../_images/go_summary_results.jpg" src="../../_images/go_summary_results.jpg" />
<p>An interactive bar plot shows each term frequency. Click on a bar to display the proteins mapped to the corresponding term. Each protein group can also be viewed by browsing the list of terms displayed below the plot.</p>
</div>
<div class="section" id="go-enrichment-analysis">
<h3>GO enrichment analysis<a class="headerlink" href="#go-enrichment-analysis" title="Permalink to this headline">¶</a></h3>
<p>Enrichment analysis is performed to determine which GO terms are significantly enriched in a <strong>tested set</strong> of proteins when compared with a given <strong>background set</strong> (eg. the whole proteome of the species studied). All terms will be tested regardless of their depth.
In myProMS, GO enrichment analysis is calculated with the GO::TermFinder package developed for perl<a class="footnote-reference" href="#id3" id="id1">[1]</a>. Briefly, a p-value using a hypergeometric distribution is computed to determine whether any GO terms annotate a specified list of proteins at a frequency greater than that would be expected by chance. Multiple hypothesis correction is available with FDR computing.
This tool is accessible by clicking on an experiment and selecting the <code class="docutils literal notranslate"><span class="pre">Start</span> <span class="pre">GO</span> <span class="pre">Analysis</span></code> button in the option frame. The following form in then displayed:</p>
<img alt="../../_images/go_settings.jpg" src="../../_images/go_settings.jpg" />
<table border="1" class="docutils">
<colgroup>
<col width="3%" />
<col width="5%" />
<col width="92%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td>Name</td>
<td colspan="2">Provide a name for the enrichment analysis. The analysis is saved and can be retrieved by this name in the GO analyses tree displayed in the sub-navigation frame</td>
</tr>
<tr class="row-even"><td>Description (Optional)</td>
<td colspan="2">Description of the current analysis</td>
</tr>
<tr class="row-odd"><td>Ontology file</td>
<td colspan="2">The file containing term relationships</td>
</tr>
<tr class="row-even"><td>Domain(s)</td>
<td colspan="2">Select one or more domains to test</td>
</tr>
<tr class="row-odd"><td rowspan="5">Advanced parameters
(Optional)</td>
<td>Estimated number of proteins in organism</td>
<td>If the background population consists of the whole proteome (more exactly the whole protein set contained in the annotation file), this value can be set to calculate properly the enrichment ratio of GO terms in the tested  protein set(s), supposing that the annotation file is incomplete. This option artificially adds unannotated proteins to the background</td>
</tr>
<tr class="row-even"><td>Background population</td>
<td>Select the population to which the tested protein set will be compared. A previously built custom list can be selected, or a local file can be used instead. This file must contains all protein identifiers that compose the background (1 identifier per row). These identifiers must match the ones contained in the annotation file. If selected background is set to “Unspecified”, the whole protein set contained in the annotation file will be used as background. In this case, be sure that the annotation file contains only proteins from the current species. This can be considered as a whole proteome background if the annotation has a very good coverage of current species proteome. The background population selection strongly affects the significance of terms and must be chosen carefully and coherently with your biological question</td>
</tr>
<tr class="row-odd"><td>Statistical settings</td>
<td>These settings can be set to control the significance cut-off of GO terms. False Discovery Rate (FDR) or p-value criteria can be selected.</td>
</tr>
<tr class="row-even"><td>Show non-significant terms in graph</td>
<td>If this option is disabled, non-significant terms will be represented with small dots in graphical view. This can increase significantly the visibility of the graph if the dataset contains a large number of significant terms</td>
</tr>
<tr class="row-odd"><td>Include only proteins with at least n peptide(s)</td>
<td>Proteins which contains less peptides than the value specified will be excluded from the tested set</td>
</tr>
<tr class="row-even"><td>Select a protein set</td>
<td colspan="2">Select the protein set to be tested. It can be selected from any project item or custom list</td>
</tr>
</tbody>
</table>
<p>Once all parameters have been set, click on <code class="docutils literal notranslate"><span class="pre">Start</span> <span class="pre">Analysis</span></code>. The computation may last several minutes depending on the sizes of the protein sets being compared.
The results are directly displayed after the process but can also be accessed later on by selecting the analysis name in the GO analyses tree displayed in the sub-navigation frame.</p>
<img alt="../../_images/go_results.jpg" src="../../_images/go_results.jpg" />
<p>For each domain, results can be displayed in 3 different views accessible at the top of the page:</p>
<ul class="simple">
<li><strong>Cloud view</strong>: Highly significant terms (low p-value) are represented with a large font, and less significant terms with a small font. The proteins mapped to a term can be listed by clicking and each term.</li>
<li><strong>Table view</strong>: More details can be viewed in table format which contains the p-value and enrichment ratio of each term.</li>
<li>Graph view: Displays a graph of the significant terms as nodes with their relationships as edges. Each node colour is based on the corresponding term’s p-value significance. Proteins that are mapped to a term can be viewed by clicking on the corresponding node.</li>
</ul>
</div>
<div class="section" id="quantitative-gene-enrichment-analysis">
<h3>Quantitative gene enrichment analysis<a class="headerlink" href="#quantitative-gene-enrichment-analysis" title="Permalink to this headline">¶</a></h3>
<p>When a quantification is available, a quantitative gene enrichment analysis can be performed as it was originally done for SILAC experiments<a class="footnote-reference" href="#id4" id="id2">[2]</a>.
The quantified proteome is divided into five bins corresponding to log2 ratios or bin proportion. Enrichment of GO terms in each bin is then calculated compared to a provided background and a cluster analysis allow to visualize a heatmap of enriched GO-terms in all bins.
Here is how you should proceed to do it.
This option is accessible by clicking on an experiment and selecting the <code class="docutils literal notranslate"><span class="pre">Start</span> <span class="pre">Q.</span> <span class="pre">GO</span> <span class="pre">Analysis</span></code> button in the option frame. After loading a protein set of an Analysis or a Design related quantification, you need to select the parameters in the following form:</p>
<img alt="../../_images/qgo_settings.jpg" src="../../_images/qgo_settings.jpg" />
<table border="1" class="docutils">
<colgroup>
<col width="15%" />
<col width="17%" />
<col width="68%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td>Name</td>
<td colspan="2">Provide a name for the enrichment analysis. The analysis is saved and can be retrieved by this name in the GO analyses tree displayed in the sub-navigation frame</td>
</tr>
<tr class="row-even"><td>Description (Optional)</td>
<td colspan="2">Description of the current analysis</td>
</tr>
<tr class="row-odd"><td>Ratio</td>
<td colspan="2">Choose the ratio considered for the enrichment in the quantitation (like heavy/medium or heavy/light for SILAC experiments).</td>
</tr>
<tr class="row-even"><td>Peptides (Optional)</td>
<td colspan="2">Make a threshold upon the number of peptides used to compute the ratio</td>
</tr>
<tr class="row-odd"><td>Protein-ratio p-value threshold</td>
<td colspan="2">Make a selection on the associated p-value of the ratio</td>
</tr>
<tr class="row-even"><td>Ontology file</td>
<td colspan="2">The file containing term relationships</td>
</tr>
<tr class="row-odd"><td>Annotation</td>
<td colspan="2">The file containing protein annotations to GO terms</td>
</tr>
<tr class="row-even"><td>Domain</td>
<td colspan="2">Select one domain to test</td>
</tr>
<tr class="row-odd"><td rowspan="2">Advanced parameters
(Optional)</td>
<td>Background population</td>
<td>Select the population to which the tested protein set will be compared. See GO enrichment analysis section for custom list recommendations</td>
</tr>
<tr class="row-even"><td>Enrichment test statistical settings</td>
<td>These settings can be set to control the significance cut-off of GO terms. False Discovery Rate (FDR) or p-value criteria can be selected</td>
</tr>
</tbody>
</table>
<p>When the enrichment is done, you can get information of the GO-Analysis by clicking on the sub-navigation frame the item generated and Summary.</p>
<p>Click on the <code class="docutils literal notranslate"><span class="pre">Heatmap</span></code> button to see the output you can get:</p>
<img alt="../../_images/qgo_results.jpg" src="../../_images/qgo_results.jpg" />
<p>Each row represent a GO-Term and each cell is the -log10 of the p-value of the enrichment test for the GO-Term in the specific bin (put to 1 and then log-transformed to 0 if that ontology is not enriched/significant in the bin).</p>
<p>Each line is z-scored. Then, these z-scores are clustered by one-way hierarchical clustering using the function hclust in R (the distance function used is <strong>euclidean</strong> and the agglomeration method used is <strong>average</strong>).</p>
<div class="admonition note">
<p class="first admonition-title">Note</p>
<p class="last">The heatmap is interactive and can be exported as a jpeg image. Clicking on a cell updates the frame and provides the list of proteins containing the annotated GO-Term in the bin.</p>
</div>
<hr class="docutils" />
<table class="docutils footnote" frame="void" id="id3" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label"><a class="fn-backref" href="#id1">[1]</a></td><td>GO::TermFinder package : <a class="reference external" href="http://bioinformatics.oxfordjournals.org/content/20/18/3710.long">Boyle et al., Bioinformatics, 2004</a></td></tr>
</tbody>
</table>
<table class="docutils footnote" frame="void" id="id4" rules="none">
<colgroup><col class="label" /><col /></colgroup>
<tbody valign="top">
<tr><td class="label"><a class="fn-backref" href="#id2">[2]</a></td><td>Quantitative GO SILAC : <a class="reference external" href="http://www.ncbi.nlm.nih.gov/pubmed/18952599">Pan C et al, MCP, 2009</a></td></tr>
</tbody>
</table>
</div>
</div>
<div class="section" id="pathway-enrichment">
<h2>Pathway enrichment<a class="headerlink" href="#pathway-enrichment" title="Permalink to this headline">¶</a></h2>
<p>Pathway is the term from molecular biology which depicts an artificial simplified model of a process within a cell or tissue.
In bioinformatics research, <strong>pathway analysis</strong> is used to identify <strong>related proteins</strong> within a pathway.
This is helpful when analyzing any omics dataset with a large number of proteins. By examining the changes in proteins in a pathway, its biological causes can be explored.</p>
<p>Typical pathway model starts with extracellular signaling molecule that activates a specific protein.
Thus triggers a chain of protein-protein or protein-small molecule interactions.
Pathway analysis helps to understand or interpret omics data from the point of view of canonical prior knowledge structured in the form of pathways diagrams.
It allows finding distinct cell processes (cellular processes), diseases or signalling pathway that are statistically associated with selection of differentially expressed proteins between two samples.
To do so, <em>myProms</em> uses <a class="reference external" href="https://reactome.org/documentation">Reactome</a>’s web-service.</p>
<div class="section" id="launch-pathway-analysis">
<h3>Launch pathway analysis<a class="headerlink" href="#launch-pathway-analysis" title="Permalink to this headline">¶</a></h3>
<img alt="../../_images/pathway_analysis_settings.png" src="../../_images/pathway_analysis_settings.png" />
<table border="1" class="docutils">
<colgroup>
<col width="12%" />
<col width="88%" />
</colgroup>
<tbody valign="top">
<tr class="row-odd"><td>Name</td>
<td>Provide a name for the Pathway analysis. The analysis is saved and can be retrieved by this name in the Functional analyses tree displayed in the sub-navigation frame</td>
</tr>
<tr class="row-even"><td>Description
(Optional)</td>
<td>Free text</td>
</tr>
<tr class="row-odd"><td>Advanced parameters
(Optional)</td>
<td>Set a p-value threshold</td>
</tr>
<tr class="row-even"><td>Select proteins</td>
<td>Select proteins to use in the analysis, from custom list or project item</td>
</tr>
</tbody>
</table>
</div>
<div class="section" id="summary-edit-delete">
<h3>Summary / edit / delete<a class="headerlink" href="#summary-edit-delete" title="Permalink to this headline">¶</a></h3>
<img alt="../../_images/pathway_analysis_summary.png" src="../../_images/pathway_analysis_summary.png" />
<p>If a pathway analysis is selected, a summary of the information available for that analysis is displayed in the mainframe.</p>
</div>
<div class="section" id="displaying-pathway-analysis">
<h3>Displaying pathway analysis<a class="headerlink" href="#displaying-pathway-analysis" title="Permalink to this headline">¶</a></h3>
<p>3 different views are available : <strong>Cloud</strong>, <strong>graphical</strong> and <strong>table view</strong> :</p>
<p><strong>Cloud view :</strong></p>
<img alt="../../_images/pathway_analysis_results_cloud.png" src="../../_images/pathway_analysis_results_cloud.png" />
<p>The displaying pathway is ordered by p-value. The color corresponds to the enrichment factor value.</p>
<p>Users have access to the unannotated proteins, which does not match with the reactome database.</p>
<img alt="../../_images/pathway_analysis_results_unannotated.png" src="../../_images/pathway_analysis_results_unannotated.png" />
<img alt="../../_images/pathway_analysis_results_unannotated_details.png" src="../../_images/pathway_analysis_results_unannotated_details.png" />
<p><strong>Graphical view :</strong></p>
<img alt="../../_images/pathway_analysis_results_graph.png" src="../../_images/pathway_analysis_results_graph.png" />
<img alt="../../_images/pathway_analysis_results_details.png" src="../../_images/pathway_analysis_results_details.png" />
<div class="admonition danger">
<p class="first admonition-title">Danger</p>
<p class="last">A COMPLETER</p>
</div>
<p><strong>Table view :</strong></p>
<img alt="../../_images/pathway_analysis_results_table.png" src="../../_images/pathway_analysis_results_table.png" />
<p>By default, pathways are ordered by p-value. They can be sorted by enrichment factor or FDR by clicking the corresponding name.
A pathway can be display in Reactome by clicking on its name.</p>
<img alt="../../_images/pathway_analysis_reactome.png" src="../../_images/pathway_analysis_reactome.png" />
</div>
<div class="section" id="export">
<h3>Export<a class="headerlink" href="#export" title="Permalink to this headline">¶</a></h3>
<p>An excel file can be downloaded which correspond to the table view information.</p>
</div>
</div>
<div class="section" id="motif-enrichment-analysis">
<h2>Motif enrichment analysis<a class="headerlink" href="#motif-enrichment-analysis" title="Permalink to this headline">¶</a></h2>
<p>Motifs are recurring short sequence elements.
Their over-representation usually implies some functional significance.
<em>myProMS</em> uses <a class="reference external" href="https://github.com/omarwagih/rmotifx">rmotifx</a> R package for the enrichment analysis and <a class="reference external" href="https://github.com/omarwagih/ggseqlogo">ggseqlogo</a> R package for drawing the motif.</p>
<div class="section" id="launch-analysis">
<h3>Launch analysis<a class="headerlink" href="#launch-analysis" title="Permalink to this headline">¶</a></h3>
<img alt="../../_images/motif_analysis_settings.png" src="../../_images/motif_analysis_settings.png" />
<ul class="simple">
<li><strong>Name</strong>: provide a name for the PCA and/or the clustering analysis. The analysis is saved and can be retrieved by this name in the Exploratory analyses tree displayed in the sub-navigation frame.</li>
<li><strong>Modification</strong>: choose the type of modifications (phosphorylation, acetylation, methylation….).</li>
<li><strong>Foreground selection</strong>: select a quantification or a previously saved list.</li>
<li><strong>Foreground filtering</strong>:<ul>
<li>Ratio: all proteins with a ratio corresponding to the chosen value are kept.</li>
<li>Infinite ratio: exclude or not.</li>
</ul>
</li>
<li><strong>p-value</strong>: all proteins with a p-value smaller than or equal to the chosen value are kept.</li>
<li><strong>Central residue</strong>: following the type of modifications, the drop down menu is automatically generated. For instance, phosphorylation modification generate 3 type of residue (S, T, Y).</li>
<li><strong>Width</strong>: the width is the number of total characters is the motif. It should be an odd number between 3 and 35. However, choosing a motif width that is too narrow can result in the exclusion of motifs with critical longer-range dependencies and choosing a motif width that is too wide (without adjusting the significance threshold accordingly) can yield spurious motif results.</li>
<li><strong>Min. occurrence</strong>: the occurrence threshold refers to the minimum number of times you wish each of your extracted motifs to occur in the data set. An occurrence threshold of 20 usually is appropriate, although this parameter may be adjusted to yield more specific or less specific motifs. This parameter can be used to tune the specificity of motifs since motifs with greater specificity (i.e., more “fixed” positions) are expected to occur less often than those motifs with lower specificity. Users that wish to extract a maximal number of motifs should set this parameter to a low value (for example, “5”) and rely solely on the significance parameter (see step 8) to extract motifs. On occasion it may be useful to set this parameter as a fractional percent of the total number of modification sites in order to compare motifs with similar specificities across data sets that vary in size (e.g., to compare motifs from data sets of 300 and 3000 sites, one may opt to set the occurrences parameter to 5 and 50, respectively).</li>
<li><strong>Significance</strong>: The significance refers to the P-value threshold for the binomial probability. This is used for the selection of significant residue/position pairs in the motif.  t is critical to note that this value does not take into account a correction for multiple hypotheses (such as the Bonferroni correction). On any given motif-x search step there are (number of possible characters at each position) * (number of non fixed positions) hypotheses being tested. For example, in an S-centered analysis of width 15, there would be (20) * (14) = 280 hypotheses tested. To ensure an alpha-value of at least 0.05 by the Bonferroni method, one would need to divide the desired alpha-value by the total number of hypotheses tested (i.e., 0.05/280 = 0.00018).We suggest a threshold of 0.000001 to maintain a low false positive rate in standard protein motif analyses.</li>
<li><strong>Background selection</strong>: The background simply refers to the  item from which the data set was taken. This is important for accurate statistical analysis. You may choose the selected quantification or generate a random background sequence based on the probability of each amino acid within the proteome.</li>
</ul>
</div>
<div class="section" id="summary-editing-deleting">
<h3>Summary/editing/deleting<a class="headerlink" href="#summary-editing-deleting" title="Permalink to this headline">¶</a></h3>
<img alt="../../_images/motif_analysis_summary.png" src="../../_images/motif_analysis_summary.png" />
<p>If a motif enrichment analysis is selected, a summary of the information available for that analysis is displayed in the mainframe.</p>
<p>The motif name and the description can be modify by clicking the <code class="docutils literal notranslate"><span class="pre">edit</span></code> button.
The analysis can be deleted by clicking the <code class="docutils literal notranslate"><span class="pre">delete</span></code> button.</p>
<div class="admonition important">
<p class="first admonition-title">Important</p>
<p class="last">If the motif analysis is involved in a heatmap, you have to delete the heatmap before deleting the motif analysis.</p>
</div>
</div>
<div class="section" id="displaying-motif-enrichment">
<h3>Displaying motif enrichment<a class="headerlink" href="#displaying-motif-enrichment" title="Permalink to this headline">¶</a></h3>
<img alt="../../_images/motif_analysis_results.png" src="../../_images/motif_analysis_results.png" />
<ul class="simple">
<li>Motif: the logo picture made by ggseqlogo with the over-represented motif. Motif positions are labeled below the x-axis and residues are colored according to their chemical and physical properties.</li>
<li>Fold change:  is an indicator of the enrichment level of the extracted motifs. Specifically, it is calculated as (foreground matches/foreground size)/(background matches/background size).</li>
<li>Score: The “motif score” is calculated by taking the sum of the negative log probabilities used to fix each position of the motif. As such, higher motif scores typically correspond to motifs that are more statistically significant as well as more specific (i.e., greater number of fixed positions).</li>
<li>Fg. matches / Bg. matches: indicate the number of peptides containing a given motif in those respective data sets following the removal of all peptides containing previously extracted motifs. Because of this iterative “set reduction” strategy, the “foreground matches” and “background matches” statistics may be less than or equal to the total number of instances of a given motif in the whole data set.</li>
<li>Fg. size / Bg. size: indicate the total number of peptides contained in these data sets. The size of these data sets decreases as motifs are extracted (i.e., down a column) due to the fact that peptides are removed from both the foreground and background data sets following motif extraction. The total number of foreground peptides not falling into any extracted motif class can therefore be calculated as the difference between the “foreground size” and the “foreground matches” of the final motif class (e.g., 163 – 32 = 131 unclassified peptides in Figure 5). The “fold increase” statistic is an indicator of the enrichment level of the extracted.</li>
</ul>
<p>Clicking on either the links in the motif logo, brings users to the peptide data used to extract the given motifs. As such, the number of peptides found for each motif in this section corresponds exactly to the number of “foreground matches” for the motif in appropriate column of the results table.</p>
</div>
<div class="section" id="heatmap-motif-analysis">
<h3>Heatmap motif analysis<a class="headerlink" href="#heatmap-motif-analysis" title="Permalink to this headline">¶</a></h3>
<p><strong>Launch HM motif analysis</strong></p>
<img alt="../../_images/motif_analysis_heatmap_settings.png" src="../../_images/motif_analysis_heatmap_settings.png" />
<p>Select the previously saved motif analysis in the left panel, click on the <code class="docutils literal notranslate"><span class="pre">Add</span></code> button and the selected analysis are moved in the right panel to be cluster.</p>
<p><strong>Summary / Edit / Delete</strong></p>
<img alt="../../_images/motif_analysis_heatmap_summary.png" src="../../_images/motif_analysis_heatmap_summary.png" />
<p>&lt;to be completed&gt;</p>
<p><strong>Display Heatmap motif analysis</strong></p>
<img alt="../../_images/motif_analysis_heatmap_results.png" src="../../_images/motif_analysis_heatmap_results.png" />
<p>&lt;to be completed&gt;</p>
</div>
</div>
</div>


           </div>
           
          </div>
          <footer>
  
    <div class="rst-footer-buttons" role="navigation" aria-label="footer navigation">
      
      
        <a href="exploratory_analysis.html" class="btn btn-neutral float-left" title="Exploratory analysis" accesskey="p" rel="prev"><span class="fa fa-arrow-circle-left"></span> Previous</a>
      
    </div>
  

  <hr/>

  <div role="contentinfo">
    <p>
        &copy; Copyright 2019, Patrick Poullet, Valentin Sabatet, Victor Laigle

    </p>
  </div>
  Built with <a href="http://sphinx-doc.org/">Sphinx</a> using a <a href="https://github.com/rtfd/sphinx_rtd_theme">theme</a> provided by <a href="https://readthedocs.org">Read the Docs</a>. 

</footer>

        </div>
      </div>

    </section>

  </div>
  


  <script type="text/javascript">
      jQuery(function () {
          SphinxRtdTheme.Navigation.enable(true);
      });
  </script>

  
  
    
   

</body>
</html>